{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b8c389ff",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "from dotenv import load_dotenv\n",
    "import os\n",
    "\n",
    "load_dotenv()\n",
    "\n",
    "# Load env vars\n",
    "Entrez.email = os.getenv(\"ENTREZ_EMAIL\")\n",
    "Entrez.api_key = os.getenv(\"ENTREZ_API_KEY\")\n",
    "FETCH_LIMIT = int(os.getenv(\"PUBMED_FETCH_LIMIT\", 10))\n",
    "\n",
    "\n",
    "def fetch_abstracts(disease, max_articles=50) -> list:\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=disease, retmax=max_articles)\n",
    "    record = Entrez.read(handle)\n",
    "    handle.close()\n",
    "    ids = record[\"IdList\"]\n",
    "\n",
    "    abstracts = []\n",
    "    for pmid in ids:\n",
    "        summary_handle = Entrez.efetch(db=\"pubmed\", id=pmid, rettype=\"abstract\", retmode=\"text\")\n",
    "        abstracts.append(summary_handle.read())\n",
    "    return abstracts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "028dd4b7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[\"1. J Microbiol Biotechnol. 2025 Jun 19;35:e2501010. doi: 10.4014/jmb.2501.01010.\\n\\nEIF4A3-Induced Circ_0092278 Enhances Papillary Thyroid Cancer Cell Malignancy by \\nthe PI3K/Akt/mTOR Signaling Pathway.\\n\\nLiu M(1), Zhang L(1), Yan Z(1).\\n\\nAuthor information:\\n(1)Department of Endocrinology, Hubei No. 3 People's Hospital of Jianghan \\nUniversity, Wuhan 430033, Hubei, P.R. China.\\n\\nCircular RNA (circRNA) plays a regulatory role in cancer progression, but the \\nrole of circ_0092278 in papillary thyroid cancer (PTC) is unclear. In this \\nresearch, we aimed to reveal the effect of circ_0092278 on PTC progression as \\nwell as its interaction with the eukaryotic translation initiation factor 4A3 \\n(EIF4A3). The expression of circ_0092278 in PTC samples was analyzed using \\nquantitative real-time polymerase chain reaction (qRT-PCR). Cell experiments \\nwere conducted to assess the effects of circ_0092278 on PTC cell viability, \\nmigration, and invasion. Furthermore, key proteins involved in the \\nphosphatidylinositol 3-kinase/protein kinase B/mammalian target of rapamycin \\n(PI3K/AKT/mTOR) pathway were examined using western blotting. Additionally, a \\nxenograft model in nude mice was used to investigate the in vivo role of circ_ \\n0092278. The interactions between EIF4A3 and circ_0092278 were analyzed through \\nRNA-binding protein immunoprecipitation and qRT-PCR assays. Circ_0092278, \\nlocated in the cell cytoplasm, was significantly upregulated in PTC. Knockdown \\nof circ_0092278 reduced PTC cell proliferation, migration, invasion, and tumor \\ngrowth by inhibiting the PI3K/Akt/mTOR pathway. Conversely, overexpression of \\ncirc_0092278 prompted PTC cell malignancy by activating the PI3K/Akt/mTOR \\npathway. Additionally, EIF4A3 was found to bind to MINK1 (circ_0092278 linear \\ngene), thereby enhancing the expression of circ_0092278. Circ_0092278, \\ncontrolled by EIF4A3, promotes PTC progression by activating the PI3K/Akt/mTOR \\npathway. Our findings indicate that targeting the EIF4A3-circ_0092278 axis may \\nprovide a novel approach for treating PTC.\\n\\nDOI: 10.4014/jmb.2501.01010\\nPMID: 40537911\",\n",
       " '1. J Genet Couns. 2025 Jun;34(3):e70071. doi: 10.1002/jgc4.70071.\\n\\nIdentification and triage of patients with incidental germline pathogenic \\nvariants on somatic tumor profiling with a genomics module.\\n\\nHarris J(1), Horton N(1), Huelsman K(1).\\n\\nAuthor information:\\n(1)TriHealth Precision Medicine Institute, Cincinnati, Ohio, USA.\\n\\nWhen cancer patients undergo tumor profile testing with paired tumor tissue and \\ngermline samples, incidental germline pathogenic variants are identified in up \\nto 1 of 8 cases. Genetic counselors (GCs) play an important role in identifying \\nthese patients for genetic counseling referral and confirmatory germline \\ntesting. Tumor profile tests are typically ordered by medical oncologists, so a \\ncollaborative partnership with GCs supports appropriate follow-up. Manual \\nmethods to identify these patients can be time-consuming and tedious. In 2020, \\nour team implemented a genomics module in the electronic health record system \\n(EHR) and completed integration with the tumor testing laboratory. We used \\ndiscrete variants in our EHR to create a reporting workbench tool to identify \\npatients with incidental germline pathogenic variants. A protocol for triaging \\nflagged patients to traditional vs. embedded care confirmatory germline testing \\nwas developed for appropriate referral. Patients were considered eligible for GC \\nreferral and confirmatory germline testing after excluding those who were \\ndeceased, previously tested, or declined confirmatory germline testing with \\ndocumentation in the EHR. From 2020 to 2024, the GC referral rate increased from \\n27% to 100% and the confirmatory germline testing rate increased from 27% to \\n66%. Of all incidental germline pathogenic variants that underwent confirmatory \\ngermline testing, 100% were confirmed. Confirmatory germline testing did reveal \\nsecondary pathogenic variants not reported initially on tumor profile tests. We \\nfound additional pathogenic variants in 8.6% (2/23) and 9.7% (3/31) of patients \\ntested in 2023 and 2024, respectively. Between 2019 and 2024, 192 total \\nincidental germline patients were identified. The proportion of patients who \\nunderwent tumor profile testing with incidental germline findings was 7.3% \\noverall, which is concordant with previously reported study rates. Discrete \\nidentification of potential germline variants via EHR integration supported the \\nefficient triaging of patients eligible for GC referral and confirmatory \\ngermline testing, contributing to a 100% referral rate in 2024. Integration of \\nthe testing laboratory creates opportunity for further EHR application, such as \\nBest Practice Alerts and Health Maintenance Care Gaps. Building laboratory \\nintegration for discrete genomic variants increases reporting capabilities, \\npatient tracking, cascade testing, and could be applied to many broader medical \\ncontexts.\\n\\n© 2025 National Society of Genetic Counselors.\\n\\nDOI: 10.1002/jgc4.70071\\nPMID: 40537894',\n",
       " '1. Cancer Commun (Lond). 2025 Jun 19. doi: 10.1002/cac2.70044. Online ahead of \\nprint.\\n\\nUbiquitination in cancer: mechanisms and therapeutic opportunities.\\n\\nZhu S(1)(2)(3)(4)(5), Zhang X(1)(2)(3)(4)(5), Liu W(1)(2)(3)(4)(5), Zhou \\nZ(1)(2)(3)(4)(5), Xiong S(1)(2)(3)(4)(5), Li J(1)(2)(3)(4)(5), Chen \\nX(1)(2)(3)(4)(5), Peng C(1)(2)(3)(4)(5).\\n\\nAuthor information:\\n(1)Department of Dermatology, Xiangya Hospital, Central South University, \\nChangsha, Hunan, P. R. China.\\n(2)National Engineering Research Center of Personalized Diagnostic and \\nTherapeutic Technology, Xiangya Hospital, Central South University, Changsha, \\nHunan, P. R. China.\\n(3)Furong Laboratory, Central South University, Changsha, Hunan, P. R. China.\\n(4)Hunan Key Laboratory of Skin Cancer and Psoriasis, Human Engineering Research \\nCenter of Skin Health and Disease, Xiangya Hospital, Central South University, \\nChangsha, Hunan, P. R. China.\\n(5)National Clinical Research Center for Geriatric Disorders, Xiangya Hospital, \\nCentral South University, Changsha, Hunan, P. R. China.\\n\\nUbiquitination, a key post-translational modification, plays an essential role \\nin tumor biology by regulating fundamental cellular processes, such as \\nmetabolism and cell death. Additionally, it interacts with other \\npost-translational modifications, which are closely linked to tumorigenesis, \\ntumor progression, the tumor microenvironment, and the response to therapeutic \\ninterventions. Recent advancements in understanding the ubiquitination \\nmechanisms have led to significant breakthroughs, offering novel perspectives \\nand strategies for diagnosing and treating tumors. Here, we provided an overview \\nof how ubiquitination influences tumor biology, focusing on its roles in immune \\nregulation, metabolism, and its interactions with other modifications. We also \\nsummarized the clinical potential of targeting E3 ubiquitin ligases and \\ndeubiquitinases as therapeutic strategies in cancer treatment.\\n\\n© 2025 The Author(s). Cancer Communications published by John Wiley & Sons \\nAustralia, Ltd. on behalf of Sun Yat‐sen University Cancer Center.\\n\\nDOI: 10.1002/cac2.70044\\nPMID: 40537889',\n",
       " '1. Mil Med Res. 2025 Jun 19;12(1):31. doi: 10.1186/s40779-025-00617-z.\\n\\nLeveraging artificial intelligence for clinical decision support in personalized \\nstandard regimen recommendation for cancer.\\n\\nJiang YL(#)(1), Zhao G(#)(1), Wang SH(2), Li N(3).\\n\\nAuthor information:\\n(1)Clinical Trial Center, National Cancer Center/National Clinical Research \\nCenter for Cancer/Cancer Hospital, Chinese Academy of Medical Sciences and \\nPeking Union Medical College, Beijing, 100021, China.\\n(2)Clinical Trial Center, National Cancer Center/National Clinical Research \\nCenter for Cancer/Cancer Hospital, Chinese Academy of Medical Sciences and \\nPeking Union Medical College, Beijing, 100021, China. wangshuhang@cicams.ac.cn.\\n(3)Clinical Trial Center, National Cancer Center/National Clinical Research \\nCenter for Cancer/Cancer Hospital, Chinese Academy of Medical Sciences and \\nPeking Union Medical College, Beijing, 100021, China. lining@cicams.ac.cn.\\n(#)Contributed equally\\n\\nDOI: 10.1186/s40779-025-00617-z\\nPMID: 40537878\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: Not applicable. Competing \\ninterests: The authors declare that they have no competing interests.',\n",
       " \"1. Hereditas. 2025 Jun 19;162(1):110. doi: 10.1186/s41065-025-00469-6.\\n\\nMechanism of LINC00958 in ferroptosis of breast cancer through the SRSF1/GPX4 \\naxis.\\n\\nWang S(#)(1)(2), Liu X(#)(2), Hou X(#)(3), Sun W(1), Chen J(1), Cao Y(1), Cheng \\nH(4).\\n\\nAuthor information:\\n(1)Jiangsu Key Laboratory of Experimental & Translational Non-coding RNA \\nResearch, Institute of Translational Medicine, Yangzhou University Medical \\nCollege, Yangzhou University, Room 223, Building 6, No.136 Jiangyang Middle \\nRoad, Yangzhou, 225000, Jiangsu, China.\\n(2)The Second People's Hospital of Lianyungang City (Cancer Hospital of \\nLianyungang), Lianyungang, 222002, China.\\n(3)Department of Medical Science, Yangzhou Polytechnic College, Yangzhou, \\n225000, China.\\n(4)Jiangsu Key Laboratory of Experimental & Translational Non-coding RNA \\nResearch, Institute of Translational Medicine, Yangzhou University Medical \\nCollege, Yangzhou University, Room 223, Building 6, No.136 Jiangyang Middle \\nRoad, Yangzhou, 225000, Jiangsu, China. chenghong820@163.com.\\n(#)Contributed equally\\n\\nDOI: 10.1186/s41065-025-00469-6\\nPMID: 40537877\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: >Animal experiments were conducted under the guidance of Animal \\nCare and Use Committee of Yangzhou University Medical College. Consent for \\npublication: Not applicable. Competing interests: The authors declare no \\ncompeting interests. Animal studies: Animal studies were performed in compliance \\nwith the ARRIVE guidelines. All procedures involving the use, care, and handling \\nof animals were complied with the National Institutes of Health Guide for the \\nCare and Use of Laboratory Animals [21].\",\n",
       " \"1. Res Involv Engagem. 2025 Jun 19;11(1):68. doi: 10.1186/s40900-025-00741-z.\\n\\nIncreasing community members' engagement in cancer research: the making research \\nCLEAR program.\\n\\nMelillo RJ(1), El Khoury C(1), Godbolt JB(1), Greene Q(2), Bussard KM(1)(3), \\nLeader AE(4)(5).\\n\\nAuthor information:\\n(1)Sidney Kimmel Cancer Center, Thomas Jefferson University, Philadelphia, PA, \\nUSA.\\n(2), Atlanta, GA, USA.\\n(3)Department of Pharmacology, Physiology, and Cancer Biology, Thomas Jefferson \\nUniversity, Philadelphia, PA, USA.\\n(4)Sidney Kimmel Cancer Center, Thomas Jefferson University, Philadelphia, PA, \\nUSA. amy.leader@jefferson.edu.\\n(5)Division of Population Science, Department of Medical Oncology, Thomas \\nJefferson University, Philadelphia, PA, USA. amy.leader@jefferson.edu.\\n\\nBACKGROUND: Despite the importance of engaging community members in research, \\nmultiple barriers from the perspective of the investigator and community member \\nexist. The Making Research CLEAR (Community Learning and Experience about \\nResearch) Program aims to bridge the gap between research conducted at an \\nNCI-designated cancer center and the community.\\nMETHODS: Community members and a cancer center investigator were paired for a \\n6-month immersive research experience. Community members had at least one \\nresearch experience per month with their investigator, while also attending \\ncancer center activities such as Institutional Review Board (IRB) meetings, \\nresearch lectures, protocol review meetings, and community outreach events. \\nCommunity members and investigators completed baseline and endpoint surveys to \\nassess perceptions and impact of the program.\\nRESULTS: Seven community member-investigator dyads completed the program. There \\nwas strong agreement among community members that participation increased their \\nawareness (Pre M\\u2009=\\u20094.0, Post M\\u2009=\\u20094.43) and understanding (Pre M\\u2009=\\u20093.43, Post \\nM\\u2009=\\u20094.57) of current cancer research efforts, although there was decreased \\nagreement that their participation would add value to the research (Pre \\nM\\u2009=\\u20094.71, Post M\\u2009=\\u20094.57). Community members had increased interest in \\nparticipating in clinical trials (Pre M\\u2009=\\u20094.29, Post M\\u2009=\\u20094.43) and their \\nattitudes towards trials improved (Pre M\\u2009=\\u20094.57, Post M\\u2009=\\u20094.71). Their \\nconfidence in the safety and benefit of clinical trials was high at baseline \\n(M\\u2009=\\u20094.71) yet remained unchanged at endpoint. Researchers strongly agreed that \\nthe program offered an opportunity to build trust between researchers and \\ncommunity members (Pre M\\u2009=\\u20094.57, Post M\\u2009=\\u20095.00) and it improved their \\nunderstanding of the community's culture (Pre M\\u2009=\\u20094.57, Post M\\u2009=\\u20094.86).\\nCONCLUSION: Overall, measures of community members' trust and transparency in \\nresearch and clinical trials increased yet perceptions of their impact on \\nresearch were mixed. Community input may have limited ability to directly impact \\nthe scientific process in a span of 6 months, but researchers' willingness to \\npartner with community members in the future is promising.\\n\\nPlain Language Summary: The Making Research CLEAR program, launched by the \\nSidney Kimmel Cancer Center in Philadelphia, aims to build trust between cancer \\nresearchers and community members by involving the public directly in the \\nresearch process. Trust is especially crucial in cancer research, where sharing \\npersonal health data is often required. The program gives community members \\nhands-on lab experiences and fosters open dialogue about cancer research. \\nInitially piloted in 2022, CLEAR paired community members with researchers for \\n8–12 weeks. Based on participant feedback, the 2023 version expanded to six \\nmonths and included better training materials. Participants, often local leaders \\nor advocates, were matched with researchers based on shared interests. They took \\npart in lab work, monthly meetings, and educational events focused on topics \\nlike clinical trials and research ethics. Evaluations showed that community \\nmembers gained a better understanding of cancer research and felt more confident \\nin it. Most said the program had a positive impact and would recommend it. \\nHowever, some questioned whether their input truly influenced researchers or \\nbuilt lasting trust. Researchers, on the other hand, strongly valued community \\ninvolvement, noting improved communication skills and a greater desire to \\ncollaborate with the public. Despite some differing views, both community \\nmembers and researchers saw CLEAR as a beneficial initiative. The program \\nunderscores the need for open communication between scientists and the \\ncommunity, with the goal of improving both trust and cancer research outcomes. \\nFuture improvements may focus on fostering shared dialogue to strengthen these \\nrelationships.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s40900-025-00741-z\\nPMID: 40537862\\n\\nConflict of interest statement: Declarations. Ethical approval and consent to \\nparticipate: The study was approved by the Institutional Review Board at Thomas \\nJefferson University. Both community members and investigators provided verbal \\nconsent to participate in the research. Consent for publication: Not applicable. \\nCompeting interests: The authors declare no competing interests.\",\n",
       " \"1. Cardiooncology. 2025 Jun 19;11(1):55. doi: 10.1186/s40959-025-00354-1.\\n\\nImportance of E/e' and BNP for early detection of late cardiotoxicity in \\nlong-term follow-up of childhood hematologic cancer survivors: a retrospective \\ncross-sectional study.\\n\\nOkazoe-Hirakawa Y(1), Yakushijin K(2), Kurata K(3), Matsumoto S(3), Ichikawa \\nH(3), Sakai R(3), Nose T(3), Kimbara S(3), Nagatani Y(3), Koyama T(3), Inui \\nY(3), Funakoshi Y(3), Kiyota N(3), Ito M(3), Hatazawa K(4), Tanaka H(4), \\nYamamoto N(5), Minami H(3).\\n\\nAuthor information:\\n(1)Division of Medical Oncology/Hematology, Department of Medicine, Kobe \\nUniversity Hospital and Graduate School of Medicine, 7-5-2 Kusunoki-cho, \\nChuo-ku, Kobe, Hyogo, Japan. hirakawa@med.kobe-u.ac.jp.\\n(2)Division of Medical Oncology/Hematology, Department of Medicine, Kobe \\nUniversity Hospital and Graduate School of Medicine, 7-5-2 Kusunoki-cho, \\nChuo-ku, Kobe, Hyogo, Japan. kyakushi@med.kobe-u.ac.jp.\\n(3)Division of Medical Oncology/Hematology, Department of Medicine, Kobe \\nUniversity Hospital and Graduate School of Medicine, 7-5-2 Kusunoki-cho, \\nChuo-ku, Kobe, Hyogo, Japan.\\n(4)Division of Cardiovascular Medicine, Department of Medicine, Kobe University \\nHospital and Graduate School of Medicine, 7-5-2 Kusunoki-cho, Chuo-ku, Kobe, \\nHyogo, Japan.\\n(5)Department of Pediatrics, Kobe University Graduate School of Medicine, 7-5-2 \\nKusunoki-cho, Chuo-ku, Kobe, Hyogo, Japan.\\n\\nBACKGROUND: Childhood cancer survivors (CCS) often develop late complications \\nafter their primary disease is cured. Cardiovascular disease is one of the most \\nfrequent and serious complications that significantly affects prognosis and \\nquality of life. Early detection and appropriate intervention are expected to \\nimprove their prognosis. However, the risk factors for late cardiotoxicity in \\nCCS are not well defined, and biomarkers that can detect cardiac dysfunction \\nprior to the development of heart failure have not yet been established.\\nMETHODS: Medical records of childhood hematologic cancer survivors referred to \\nour department for transitional care between January 2016 and October 2023 were \\nreviewed for this cross-sectional study. The relationships between the most \\nrecent cardiac function at the review and history of cancer treatment were \\nanalyzed.\\nRESULTS: This study included 34 patients and the median elapsed time since \\ncancer diagnosis was 16.5 years (range, 5-30 years). None of the patients had \\nsymptomatic cardiac complications. The E/e' ratio was significantly higher in \\nsurvivors with a history of hematopoietic stem cell transplantation (HSCT) than \\nin those who did not undergo HSCT (median, 8.4% vs. 6.25%, P\\u2009=\\u20090.040), while no \\nintergroup differences were observed in ejection fraction (EF), global \\nlongitudinal strain (GLS), or the brain natriuretic protein (BNP) level. In \\naddition, the E/e' ratio was positively correlated with years elapsed since \\ncancer diagnosis (ρ\\u2009=\\u20090.38, P\\u2009=\\u20090.034). While there was no clear correlation \\nbetween years since cancer diagnosis and the BNP level in the overall cohort, a \\nstrong correlation was found in patients with a history of HSCT (ρ\\u2009=\\u20090.73; \\nP\\u2009<\\u20090.01). No significant differences were observed in EF, E/e' ratio, GLS, and \\nBNP level by cumulative anthracycline dose or history of chest irradiation.\\nCONCLUSIONS: In this study, no patient had late symptomatic cardiac \\ncomplications. However, in those who had survived for a long time since their \\ncancer diagnosis, particularly those with a history of HSCT, there were \\nsignificant elevations in the E/e' ratio and the BNP level. Continuous follow-up \\nis required to determine whether these abnormalities lead to symptomatic \\ncardiotoxicity and whether they serve as useful markers for the early detection \\nof cardiac complications.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s40959-025-00354-1\\nPMID: 40537853\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: This study was approved by the Ethics Committee of Kobe University \\nHospital (No. B240214). The requirement for informed consent was waived due to \\nthe retrospective nature of the study. The patients who participated in this \\nstudy were offered the opportunity to opt-out. Consent for publication: Not \\napplicable. Competing interests: The authors declare no competing interests.\",\n",
       " \"1. J Med Case Rep. 2025 Jun 19;19(1):285. doi: 10.1186/s13256-025-05152-y.\\n\\nStruma ovarii-diagnostic and treatment strategy: a case report.\\n\\nAlnaggar M(1)(2), Mahyoub MA(3)(4), Alshargabi N(5), Gong L(6).\\n\\nAuthor information:\\n(1)Department of Oncology, South Hubei Cancer Hospital, Chibi, Xinning City, \\nHubei Province, China. dr.alnaggar@hotmail.com.\\n(2)Department of Internal Medicine, The Maternity and Childhood Teaching \\nHospital, Ibb, Yemen. dr.alnaggar@hotmail.com.\\n(3)Department of Gastroenterology, First Affiliated Hospital of Xi'an Jiaotong \\nUniversity, Xi'an City, Shaanxi Province, China.\\n(4)Clinical Medical Research Center for Digestive Diseases (Oncology) of Shaanxi \\nProvince, Xi'an, China.\\n(5)School of Medicine, Almahwit University, Almahwait, Yemen.\\n(6)Department of Oncology, South Hubei Cancer Hospital, Chibi, Xinning City, \\nHubei Province, China. 273971995@qq.com.\\n\\nINTRODUCTION: Struma ovarii is an uncommon ovarian tumor primarily composed of \\nthyroid tissue, accounting for approximately 1-3% of ovarian teratomas.\\nCASE PRESENTATION: We present the case of a 68-year-old Chinese female patient \\npresenting with recurrent urethral outlet tumescence and an adnexal mass, whose \\ndiagnosis was established through ultrasonography, pelvic computed tomography, \\nand magnetic resonance imaging. Subsequently, the patient underwent laparoscopic \\nleft adnexectomy, and histopathological examination confirmed the presence of \\nstruma ovarii in the left ovary.\\nCONCLUSION: Struma ovarii poses unique diagnostic and management challenges \\nowing to its rarity. A multidisciplinary approach incorporating clinical, \\nimaging, and histological evaluations is essential for accurate diagnosis and \\noptimal treatment outcomes. Surgical excision remains the cornerstone of \\ntreatment, typically resulting in a favorable prognosis when performed promptly \\nand appropriately.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s13256-025-05152-y\\nPMID: 40537851\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Written informed consent was obtained from the patients for \\nparticipation in this study. Consent for publication: Written informed consent \\nwas obtained from the patient for publication of this case report and any \\naccompanying images. A copy of the written consent is available for review by \\nthe Editor-in-Chief of this journal. Competing interests: The authors declare \\nthat they have no known competing financial interests or personal relationships \\nthat could have appeared to influence the work reported in this paper.\",\n",
       " '1. Clin Epigenetics. 2025 Jun 19;17(1):105. doi: 10.1186/s13148-025-01913-0.\\n\\nEpigenetic editing and epi-drugs: a combination strategy to simultaneously \\ntarget KDM4 as a novel anticancer approach.\\n\\nSarno F(1)(2), Jacob JJ(1), Eilers RE(1), Nebbioso A(2)(3), Altucci L(2)(4)(3), \\nRots MG(5).\\n\\nAuthor information:\\n(1)Epigenetic Editing Research Group, Department of Pathology and Medical \\nBiology, University of Groningen, University Medical Center Groningen, \\nHanzeplein 1, 9713 GZ, Groningen, The Netherlands.\\n(2)Dipartimento Di Medicina Di Precisione, Università Degli Studi Della Campania \\n\"Luigi Vanvitelli\" Napoli, Naples, Italy.\\n(3)Medical Epigenetics Program, Azienda Ospedaliera Universitaria \"Luigi \\nVanvitelli\", Naples, Italy.\\n(4)Biogem, Molecular Biology and Genetics Research Institute, Ariano Irpino, \\nItaly.\\n(5)Epigenetic Editing Research Group, Department of Pathology and Medical \\nBiology, University of Groningen, University Medical Center Groningen, \\nHanzeplein 1, 9713 GZ, Groningen, The Netherlands. m.g.rots@umcg.nl.\\n\\nKDM4-A/B/C, preferentially demethylating di- and tri-methylated lysine 9 on \\nhistone H3, are overexpressed in cancers and considered interesting therapeutic \\ntargets. Consequently, KDM4 inhibitors have been developed to block their \\nenzymatic activity. However, the potential lack of specificity of such small \\nmolecules (epi-drugs) may contribute to dose-limiting toxicities. In the pursuit \\nof more specific interventions, epigenetic editing (epi-editing) has emerged as \\na powerful tool to modulate gene expression by modifying the epigenetic profile \\nof specific genomic locations. The recently developed CRISPRoff (dCas9 fused to \\nDNMT3A/3L and KRAB), guided by sgRNAs, is successfully used for gene repression \\nby introducing methylation of DNA and (indirectly) of histones at the targeted \\ngenomic region. We propose that combining epi-editing (here to prevent the \\nexpression of KDM4) with epi-drugs (to inhibit the KDM4 protein activity) may \\nrepresent a novel path for synergistic anticancer effects through simultaneous \\ninhibition of gene expression and protein activity. Upon validating the \\ndownregulation of KDM4A in HEK293T cells through epi-editing, we demonstrated \\nits repression in colon, breast and hepatocellular carcinomas which was \\neffective in preventing (breast, MCF7) or inhibiting (colon, HCT116) cancer cell \\ngrowth. Anticancer effect was also confirmed for these cell lines using the KDM4 \\ninhibitor QC6352. In parallel, our studies demonstrate a previously unnoticed \\nincrease in the expression of KDM4-A/B/C genes following the inhibition of \\nprotein activity using the pan-KDM4 inhibitors QC6352 and JIB-04. Importantly, \\nthis induction of gene expression was fully prevented or even further inhibited \\nby epi-editing. Then, we assessed the efficacy of our dual-targeted silencing \\napproach in cancer cells and demonstrated that the inhibition in cancer cell \\ngrowth by epi-drug or epigenetic editing could be further improved by combining \\nthe treatments. Building upon these findings, we introduce a novel, potentially \\nsynergistic, therapeutic strategy that combines epi-drug administration with \\nepi-editing. This innovative approach aims to reduce drug toxicity and the \\npotential development of resistance by preventing drug-induced upregulation of \\ntarget enzyme expression, thereby further increasing anticancer effects.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s13148-025-01913-0\\nPMID: 40537846\\n\\nConflict of interest statement: Declarations. Conflict of interests: The authors \\ndeclare no competing interests. Consent for publication: Not applicable. Ethics \\napproval: Not applicable.',\n",
       " \"1. Reprod Biol Endocrinol. 2025 Jun 19;23(1):91. doi: 10.1186/s12958-025-01427-7.\\n\\nBEZ235-Mediated PI3K/mTOR dual inhibition improves ovarian follicle survival in \\na preclinical model.\\n\\nBindels J(1), Bernet L(1), Squatrito M(1), Nisolle M(2), Munaut C(3).\\n\\nAuthor information:\\n(1)Laboratory of Biology of Tumor and Development, GIGA-Cancer, Université de \\nLiège, 4000, Liège, Belgium.\\n(2)Department of Obstetrics and Gynecology, Hôpital de La Citadelle, Université \\nde Liège, 4000, Liège, Belgium.\\n(3)Laboratory of Biology of Tumor and Development, GIGA-Cancer, Université de \\nLiège, 4000, Liège, Belgium. c.munaut@uliege.be.\\n\\nBACKGROUND: Follicular loss after ovarian tissue cryopreservation and \\nautotransplantation (OTCTP) remains a major challenge due to follicle activation \\nand ischemia. We evaluated BEZ235, a dual PI3K/mTOR inhibitor, as a strategy to \\nimprove follicle survival in a preclinical model. Its effects were evaluated \\nduring ovarian culture, cryopreservation, and transplantation, including the \\npotential benefit of post-grafting VEGF/G-CSF injections.\\nMETHODS: Murine ovaries, organotipically cultured with chemotherapeutic \\ntreatment (4-HC, 2\\xa0µM), with or without supplementation with BEZ235 (1\\xa0µM), \\nrapamycin (1\\xa0µM), LY294002 (25\\xa0µM), or AMH (200\\xa0ng/ml) were used to evaluate \\nfollicle activation. For cryopreservation studies, those inhibitors were added \\nto the freezing medium, and pathways activation were assessed via Western blot. \\nIn vivo, ovaries cryopreserved with or without BEZ235 or rapamycin were \\nautotransplanted under the kidney capsule of mice. A subset of mice received \\nintraperitoneal VEGF/G-CSF injections for five days post-transplantation. \\nFollicle quantification, proliferation and activation marker assessment, and \\nfibrosis evaluation were performed three weeks post-grafting.\\nRESULTS: In vitro, BEZ235 significantly counteracted chemotherapy-induced \\nactivation of both Akt and mTOR pathways, whereas rapamycin and LY29400 \\ninhibited only mTOR or Akt, respectively. Similarly, during cryopreservation, \\nonly BEZ235 significantly reduced activation of both pathways. AMH did not \\nenhance BEZ235's protective effects. In vivo, ovaries slow-frozen with BEZ235 \\nretained a higher percentage of primordial follicles and showed reduced follicle \\nproliferation and activation compared to both control and rapamycin three weeks \\nafter transplantation. Additionally, post-grafting injection of VEGF/G-CSF did \\nnot further enhance follicle preservation or reduce fibrosis.\\nCONCLUSION: Dual inhibition of PI3K/mTOR with BEZ235 provides superior \\nprotection of the primordial follicle pool by maintaining follicle dormancy, in \\nboth in vitro and in vivo models. These findings highlight BEZ235's potential to \\nenhance OTCTP outcomes, extend graft longevity and improve fertility \\npreservation strategies in women.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12958-025-01427-7\\nPMID: 40537830\\n\\nConflict of interest statement: Declarations. Ethical approval and consent to \\nparticipate: This study was approved by the Animal Ethics Committee of the \\nUniversity of Liège (#1934 and #2594). We confirmed that all experiments in this \\nstudy were performed in accordance with the relevant guidelines and regulations. \\nAll the procedure of the study is followed by the ARRIVE guidelines. Consent of \\npublication: Not applicable. Competing interest: The authors declare no \\ncompeting interests.\",\n",
       " \"1. Breast Cancer Res. 2025 Jun 19;27(1):111. doi: 10.1186/s13058-025-02014-9.\\n\\nHope for OTHERS (Our Tissue Helping Enhance Research & Science): research \\nresults from the University of Pittsburgh rapid autopsy program for breast \\ncancer.\\n\\nChang AC(1)(2), Balic M(1)(2)(3), Bartholow T(1)(4), Bhargava R(1)(4), Brown \\nDD(1)(5), Brown L(1)(2), Brufsky A(1)(2), Cao Y(1)(2), Carleton N(1)(2), Clark \\nAM(1)(4), Cody M(1)(2), Ding K(1)(2), Deible C(1)(2)(6), Elangovan A(1)(2), \\nFoldi J(1)(2), Geisler D(6), Hodgdon C(7), Howard N(8), Li Z(1)(2), Liu \\nJB(1)(2), Lopez-Nunez O(4)(9), Mary DJSJ(1)(2), McGinn O(1)(2), Miller L(1), \\nMori K(1)(2), Pecar G(1)(2), Priedigkeit N(10), Puhalla S(1)(2)(3), Rosenzweig \\nMQ(1)(11), Roy P(1)(12), Savariau L(1)(2), Walker S(13)(14), Waltermire H(1)(2), \\nWedn AM(1)(15), Wells A(1)(4), Yates ME(1)(2), Xavier J(1)(2), Lee AV(1)(15)(5), \\nOesterreich S(16)(17)(18).\\n\\nAuthor information:\\n(1)Womens Cancer Research Center, UPMC Hillman Cancer Center, Magee-Women \\nResearch Institute, Pittsburgh, PA, USA.\\n(2)Department of Medicine, UPMC, Pittsburgh, PA, USA.\\n(3)NSABP, Pittsburgh, PA, USA.\\n(4)Department of Pathology, Magee Women's Hospital, Pittsburgh, PA, USA.\\n(5)Institute of Precision Medicine, Pittsburgh, PA, USA.\\n(6)Department of Radiology, UPMC, Pittsburgh, PA, USA.\\n(7)GRASP, Baltimore, MD, USA.\\n(8)bcRAN, Pittsburgh, PA, USA.\\n(9)Cincinnati Children's Hospital Medical Center, Cincinnati, OH, USA.\\n(10)DFCI/Broad Institute of MIT and Harvard, Boston, MA, USA.\\n(11)School of Nursing, University of Pittsburgh, Pittsburgh, PA, USA.\\n(12)School of Bioengineering, Pittsburgh, PA, USA.\\n(13)MBC Alliance, New York City, NY, USA.\\n(14)LBBC, Philadelphia, PA, USA.\\n(15)Department of Pharmacology and Chemical Biology, University of Pittsburgh, \\nPittsburgh, PA, USA.\\n(16)Womens Cancer Research Center, UPMC Hillman Cancer Center, Magee-Women \\nResearch Institute, Pittsburgh, PA, USA. Oesterreichs@upmc.edu.\\n(17)Department of Pharmacology and Chemical Biology, University of Pittsburgh, \\nPittsburgh, PA, USA. Oesterreichs@upmc.edu.\\n(18)School of Nursing, University of Pittsburgh, Pittsburgh, PA, USA. \\nOesterreichs@upmc.edu.\\n\\nBreast cancer affects 1/8 of women throughout their lifetimes, with over 90% of \\ncancer deaths being caused by metastasis. However, metastasis poses unique \\nchallenges to research, as complex changes in the microenvironment in different \\nmetastatic sites and difficulty obtaining tissue for study hinder the ability to \\nexamine in depth the changes that occur during metastasis. Rapid autopsy \\nprograms thus fill a unique need in advancing metastasis research. Here, we \\ndescribe our protocol and processes for establishing and improving the US-based \\nHope for OTHERS (Our Tissue Helping Enhance Research and Science) program for \\norgan donation in metastatic breast cancer. As of August 2024, we consented 114 \\npatients and performed 37 autopsies, from which we collected 551 unique \\nmetastatic frozen tumor samples, 1244 FFPE blocks, 90 longitudinal liquid biopsy \\nsamples and developed 14 patient-derived organoid and 8 patient-derived \\nxenograft models. We report in-depth clinical and histopathological information \\nand discuss extensive new research and novel findings in patient outcomes, \\nmetastatic phylogeny, and factors in successful living model development. Our \\nresults reveal key logistical and protocol improvements that are uniquely \\nbeneficial to certain programs based on identifiable features, such as working \\nclosely with patient advocates, methods to rescue RNA quality in cases where \\ntissue quality may degrade due to time delays, as well as guidelines and future \\nexpansions of our program.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s13058-025-02014-9\\nPMID: 40537829\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: All research involving human participants was conducted with \\ninformed consent obtained according to the ethical guidelines of the University \\nof Pittsburgh Institutional Review Board under STUDY19060376. Consent for \\npublication: All images of individuals on Fig.\\xa03 are from a public website and \\nare of authors who consent to this publication. Competing interests: The authors \\ndeclare no competing interests.\",\n",
       " \"1. J Transl Med. 2025 Jun 19;23(1):688. doi: 10.1186/s12967-025-06745-3.\\n\\nDevelopment of a prognostic immune cell-based model for ovarian cancer using \\nmultiplex immunofluorescence.\\n\\nLi S(1), Jiang B(2), Zhou H(2), Yang S(2), Yang L(2), Hong Y(3).\\n\\nAuthor information:\\n(1)Ambulatory Surgery Center, Women's Hospital, Zhejiang University School of \\nMedicine, Hangzhou, China.\\n(2)Cancer Center, Department of Medical Oncology, Zhejiang Provincial People's \\nHospital(Affiliated People's Hospital), Hangzhou Medical College, Hangzhou, \\nZhejiang, China.\\n(3)Cancer Center, Department of Medical Oncology, Zhejiang Provincial People's \\nHospital(Affiliated People's Hospital), Hangzhou Medical College, Hangzhou, \\nZhejiang, China. hypbdn@zju.edu.cn.\\n\\nBACKGROUND: Ovarian cancer is the most lethal gynecological malignancy, often \\ndiagnosed at advanced stages with poor prognosis. The tumor microenvironment \\n(TME) plays a critical role in disease progression and treatment response. This \\nstudy aimed to construct a prognostic model for ovarian cancer patients by \\nevaluating the tumor immune landscape using multiplex immunofluorescence (mIF) \\nstaining, which focused on the spatial distribution and interactions of immune \\ncells within the TME.\\nMETHODS: Formalin-fixed paraffin-embedded (FFPE) tissues from 129 ovarian cancer \\npatients were analyzed using mIF to assess the expression of PD-L1(Programmed \\ndeath-ligand 1, PD-L1), CD8(Cluster of Differentiation 8, CD8), TOX (Thymocyte \\nSelection-Associated HMG Box, TOX), CD68(Cluster of Differentiation 68, CD68), \\nand CK (Cytokeratin, CK). The Vectra Polaris quantitative pathology imaging \\nsystem and Inform software were employed for image and spatial analysis. The \\nLASSO Cox regression model was used for feature selection, and Kaplan-Meier \\nsurvival analysis was performed to evaluate the prognostic significance of \\nimmune cell markers. A nomogram was developed to predict overall survival (OS) \\nbased on clinical parameters and the Immune Cell Related Prognostic Index \\n(ICRPI).\\nRESULTS: High percentages of CD8\\u2009+\\u2009T cells, CD68\\u2009+\\u2009macrophages, and \\nCD68\\u2009+\\u2009PD-L1\\u2009+\\u2009macrophages were significantly associated with poor OS. Moreover, \\na high percentage of CD8\\u2009+\\u2009T cells, CD68\\u2009+\\u2009macrophages was significantly \\nassociated with poor disease-free survival (DFS). Spatial analysis revealed that \\na higher average count of CD68\\u2009+\\u2009PD-L1\\u2009+\\u2009macrophages within 30\\xa0μm of CD8\\u2009+\\u2009T \\ncells correlated with worse prognosis. The ICRPI model, incorporating CD68+, \\nCD68\\u2009+\\u2009PD-L1+, and spatial variables, effectively stratified patients into high- \\nand low-risk groups, with high-risk patients showing significantly poorer OS.\\nCONCLUSION: This study highlights the prognostic value of immune cell spatial \\ndistribution in ovarian cancer. The ICRPI model, integrating immune cell markers \\nand spatial analysis, provides a novel framework for predicting patient \\noutcomes. Further validation in prospective studies is warranted to confirm the \\nclinical utility of this model.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12967-025-06745-3\\nPMID: 40537823\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: All procedures conducted in this study using human data were in \\naccordance with the Declaration of Helsinki. This study involving human \\nparticipants were reviewed and approved by the human ethics committees of \\nZhejiang Provincial People’s Hospital, Hangzhou medical college (Approval \\nnumber: QT2024015) and the Ethical Board of the Institutional Review Board of \\nShanghai Outdo Biotech Company (Approval number: YB M 05\\u2009−\\u200902). Consent for \\npublication: Not applicable. Competing interests: The authors declare that the \\nresearch was conducted in the absence of any commercial or financial \\nrelationships that could be construed as a potential conflict of interest.\",\n",
       " '1. Cell Commun Signal. 2025 Jun 19;23(1):292. doi: 10.1186/s12964-025-02225-9.\\n\\nRole of DEAD/DEAH-box helicases in immunity, infection and cancers.\\n\\nDevasahayam Arokia Balaya R(1), Kanekar S(2), Kumar S(3), Kandasamy RK(4)(5)(6).\\n\\nAuthor information:\\n(1)Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN, \\n55905, USA.\\n(2)Centre for Integrative Omics Data Science, Yenepoya (Deemed to Be \\nUniversity), Mangalore, Karnataka, 575018, India.\\n(3)Department of Biological Sciences, Sunandan Divatia School of Science, Narsee \\nMonjee Institute of Management Studies, Mumbai, Maharashtra, 400056, India.\\n(4)Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN, \\n55905, USA. Kandasamy.RichardKumaran@Mayo.edu.\\n(5)Department of Immunology, Mayo Clinic, Rochester, MN, 55905, USA. \\nKandasamy.RichardKumaran@Mayo.edu.\\n(6)Department of Quantitative Health Sciences, Mayo Clinic, Rochester, MN, \\n55905, USA. Kandasamy.RichardKumaran@Mayo.edu.\\n\\nDEAD/DEAH-box helicases (DDX) are integral RNA-binding proteins within the RNA \\nhelicase superfamily 2 (SF2), characterized by distinct DEAD (Asp-Glu-Ala-Asp) \\nand DEAH (Asp-Glu-Ala-His) motifs. These motifs delineate two subfamilies: \\nDEAD-box (DDX) and DEAH-box (DHX). DEAD-box proteins predominantly facilitate \\nlocalized non-processive RNA duplex destabilization, whereas DEAH-box helicases \\nmediate processive RNA translocation and unwinding. This functional dichotomy is \\nattributed to Asp-to-His substitution in the DEAH motif, which modulates ATP \\nhydrolysis and conformational dynamics. DEAD-box helicases have been implicated \\nin critical cellular processes, including translation, splicing, and RNA decay. \\nIn contrast, DEAH-box proteins play pivotal roles in splicing, ribosome \\nbiogenesis, and RNA export. DEAD/DEAH-box helicases play crucial roles in \\nvarious cellular processes, and their regulation is primarily governed by \\npost-translational modifications (PTMs) and protein-protein interactions (PPIs), \\nparticularly within their N- and C-terminal sequences. Despite extensive \\nresearch, significant knowledge gaps persist regarding their regulation, \\ncofactor roles, substrates, PPIs, mutation effects, and involvement in signaling \\ncascades. Mutations in DEAD domains have been associated with dysregulated \\nimmune signaling and have been implicated in various cancers, underscoring their \\nimportance in disease pathogenesis. Specific helicases, including DDX3, DDX5, \\nDDX6, and DDX41, have been extensively studied for their roles in immune \\nresponse regulation, antiviral defense, and cellular stress response. This \\nreview critically examines the DEAD-box helicases involved in cell cycle \\nregulation and their inhibitors, as well as those that regulate the Toll-like \\nreceptor signaling pathway. Furthermore, we provide comprehensive insights into \\nthe phosphorylation-based regulation of major DDX members, with a particular \\nfocus on DDX3X, DDX21, and DDX42 in various cancers. Elucidating the molecular \\nmechanisms, regulatory influences, and therapeutic potential of DEAD/DEAH-box \\nhelicases is of paramount importance, particularly in the fields of infectious \\ndiseases and immune modulation. This review provides current knowledge and \\nidentifies critical areas for future research, aiming to advance our \\nunderstanding of these essential molecular machines and their potential as \\ntherapeutic targets.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12964-025-02225-9\\nPMID: 40537811\\n\\nConflict of interest statement: Declarations. Competing interests: The authors \\ndeclare no competing interests.',\n",
       " '1. J Transl Med. 2025 Jun 19;23(1):684. doi: 10.1186/s12967-025-06566-4.\\n\\np53 in colorectal cancer: from a master player to a privileged therapy target.\\n\\nYan S(1), Zhan F(1)(2), He Y(1), Zhu Y(1), Ma Z(3)(4).\\n\\nAuthor information:\\n(1)Huzhou Key Laboratory of Molecular Medicine, Huzhou Central Hospital, The \\nFifth School of Clinical Medicine of Zhejiang, Chinese Medical University, \\nHuzhou, 313000, China.\\n(2)School of Basic Medicine College, Zhejiang Chinese Medical University, \\nHangzhou, 310000, China.\\n(3)Huzhou Key Laboratory of Molecular Medicine, Huzhou Central Hospital, The \\nFifth School of Clinical Medicine of Zhejiang, Chinese Medical University, \\nHuzhou, 313000, China. mazhihong@hzhospital.com.\\n(4)School of Basic Medicine College, Zhejiang Chinese Medical University, \\nHangzhou, 310000, China. mazhihong@hzhospital.com.\\n\\nColorectal cancer (CRC) is the third most prevalent malignancy and the second \\nleading cause of cancer-related mortality worldwide. The pathogenesis of CRC \\nprimarily stems from\\xa0the gradual accumulation of genetic mutations, which drive \\noncogene (e.g., KRAS) activation and tumor suppressor gene (e.g., TP53) \\ninactivation. Loss of genetic stability facilitates the conversion of \\nproto-oncogenes into active oncogenes and the functional impairment of tumor \\nsuppressors, collectively propelling CRC progression. The tumor suppressor \\nprotein p53, a transcription factor, induces cell cycle arrest, apoptosis, and \\nDNA damage repair under cellular stress, and prevents cancer development by \\nregulating various cellular responses. However, in CRC pathogenesis, TP53 \\nmutations (detected in\\u2009~\\u200974% of cases) subvert these protective mechanisms \\nthrough dual mechanisms: (i) dominant-negative suppression of wild-type p53 \\n(wt-p53) function, and (ii) acquisition of neomorphic pro-tumorigenic \\nactivities, termed gain-of-function (GOF) mutations. New evidence from \\nlaboratory and clinical trials shows that some new therapeutic strategies have \\nthe potential to treat CRC by reactivating and restoring p53 function, depleting \\np53 mutants, or targeting p53 with immunotherapy. In this review, we summarize \\nthe function of p53 and characterize its mutation in CRC, emphasizing the \\ninfluence of p53 mutation in the pathogenesis of CRC. In addition, we also \\ndescribe the current therapeutic strategies for targeting p53 mutants in CRC.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12967-025-06566-4\\nPMID: 40537809\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: Not applicable. Competing \\ninterests: The authors declare that they have no competing interests.',\n",
       " '1. Cell Commun Signal. 2025 Jun 19;23(1):290. doi: 10.1186/s12964-025-02308-7.\\n\\nMitochondrial dysfunction in the regulation of aging and aging-related diseases.\\n\\nZhang X(#)(1), Gao Y(#)(1), Zhang S(#)(2), Wang Y(1), Pei X(1), Chen Y(1), Zhang \\nJ(1), Zhang Y(1), Du Y(1), Hao S(3), Wang Y(4), Ni T(5)(6).\\n\\nAuthor information:\\n(1)State Key Laboratory of Reproductive Regulation and Breeding of Grassland \\nLivestock, Institutes of Biomedical Sciences, School of Life Sciences, Inner \\nMongolia University, Hohhot, 010070, China.\\n(2)Key Lab of Ministry of Education for Protection and Utilization of Special \\nBiological Resources in Western China, School of Life Sciences, Ningxia \\nUniversity, Yinchuan, 750021, China.\\n(3)State Key Laboratory of Reproductive Regulation and Breeding of Grassland \\nLivestock, Institutes of Biomedical Sciences, School of Life Sciences, Inner \\nMongolia University, Hohhot, 010070, China. shuailhao@163.com.\\n(4)Key Lab of Ministry of Education for Protection and Utilization of Special \\nBiological Resources in Western China, School of Life Sciences, Ningxia \\nUniversity, Yinchuan, 750021, China. wyj@nxu.edu.cn.\\n(5)State Key Laboratory of Reproductive Regulation and Breeding of Grassland \\nLivestock, Institutes of Biomedical Sciences, School of Life Sciences, Inner \\nMongolia University, Hohhot, 010070, China. tingni@fudan.edu.cn.\\n(6)State Key Laboratory of Genetics and Development of Complex Phenotypes, \\nNational Clinical Research Center for Aging and Medicine, Collaborative \\nInnovation Center of Genetics and Development, Center for Evolutionary Biology, \\nShanghai Engineering Research Center of Industrial Microorganisms, School of \\nLife Sciences, Huashan Hospital, Human Phenome Institute, Fudan University, \\nShanghai, 200438, China. tingni@fudan.edu.cn.\\n(#)Contributed equally\\n\\nAging is an irreversible physiological process that progresses with age, leading \\nto structural disorders and dysfunctions of organs, thereby increasing the risk \\nof chronic diseases such as neurodegenerative diseases, diabetes, hypertension, \\nand cancer. Both organismal and cellular aging are accompanied by the \\naccumulation of damaged organelles and macromolecules, which not only disrupt \\nthe metabolic homeostasis of the organism but also trigger the immune response \\nrequired for physiological repair. Therefore, metabolic remodeling or chronic \\ninflammation induced by damaged tissues, cells, or biomolecules is considered a \\ncritical biological factor in the organismal aging process. Notably, \\nmitochondria are essential bioenergetic organelles that regulate both catabolism \\nand anabolism and can respond to specific energy demands and growth repair \\nneeds. Additionally, mitochondrial components and metabolites can regulate \\ncellular processes through damage-associated molecular patterns (DAMPs) and \\nparticipate in inflammatory responses. Furthermore, the accumulation of \\nprolonged, low-grade chronic inflammation can induce immune cell senescence and \\ndisrupt immune system function, thereby establishing a vicious cycle of \\nmitochondrial dysfunction, inflammation, and senescence. In this review, we \\nfirst outline the basic structure of mitochondria and their essential biological \\nfunctions in cells. We then focus on the effects of mitochondrial metabolites, \\nmetabolic remodeling, chronic inflammation, and immune responsesthat are \\nregulated by mitochondrial stress signaling in cellular senescence. Finally, we \\nanalyze the various inflammatory responses, metabolites, and the \\nsenescence-associated secretory phenotypes (SASP) mediated by mitochondrial \\ndysfunction and their role in senescence-related diseases. Additionally, we \\nanalyze the crosstalk between mitochondrial dysfunction-mediated inflammation, \\nmetabolites, the SASP, and cellular senescence in age-related diseases. Finally, \\nwe propose potential strategies for targeting mitochondria to regulate metabolic \\nremodeling or chronic inflammation through interventions such as dietary \\nrestriction or exercise, with the aim of delaying senescence. This reviewprovide \\na theoretical foundation for organismal antiaging strategies.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12964-025-02308-7\\nPMID: 40537801\\n\\nConflict of interest statement: Declarations. Competing interests: The authors \\ndeclare no competing interests.',\n",
       " \"1. Cell Commun Signal. 2025 Jun 19;23(1):291. doi: 10.1186/s12964-025-02298-6.\\n\\nEvidence for a JAK2/STAT3 proinflammatory and vasculogenic mimicry interrelated \\nmolecular signature in adipocyte-derived mesenchymal stromal/stem cells.\\n\\nVeilleux C(1), Roy MÈ(1), Zgheib A(1), Desjarlais M(2), Annabi B(3).\\n\\nAuthor information:\\n(1)Laboratoire d'Oncologie Moléculaire, Département de Chimie and CERMO- FC, \\nUniversité du Québec à Montréal, C.P. 8888, Succ. Centreville, Montreal, QC, H3C \\n3P8, Canada.\\n(2)Department of Ophthalmology, Maisonneuve-Rosemont Hospital Research Center, \\nUniversité de Montréal, Montreal, QC, H3T 1J4, Canada.\\n(3)Laboratoire d'Oncologie Moléculaire, Département de Chimie and CERMO- FC, \\nUniversité du Québec à Montréal, C.P. 8888, Succ. Centreville, Montreal, QC, H3C \\n3P8, Canada. annabi.borhane@uqam.ca.\\n\\nBACKGROUND: During obesity, the excessive accumulation of fat in tissue promotes \\ndysregulated hormonal and cytokine homeostasis that triggers chronic \\ninflammation, which is, in part, associated with an increased incidence of some \\ncancers. This protumoral inflammatory environment is further exacerbated through \\nthe secretome of mature adipocytes, which promotes tumor angiogenesis. Emerging \\nstudies suggest that human adipocyte-derived mesenchymal stromal/stem cells \\n(ADMSCs) may contribute to a complementary process supporting local angiogenesis \\ntermed vasculogenic mimicry (VM). The molecular mechanisms linking ADMSCs to VM \\nand inflammation remain poorly understood.\\nMETHODS: ADMSC 3D capillary-like structures were generated upon seeding on \\nCultrex. Structure analysis was performed using WIMASIS. Total RNA was extracted \\nusing TRIzol and RT-qPCR was performed to assess gene expression or screen RT2 \\nPCR arrays. Transient gene silencing was performed by transfecting cells with a \\nspecific siRNA against STAT3. Protein lysates were harvested and used for \\nWestern blotting. Realtime cell migration was performed with the xCELLigence \\nsystem.\\nRESULTS: Our findings revealed that in vitro priming of ADMSCs with Cultrex led \\nto the formation of 3D capillary-like structures and the acquisition of an \\ninflammatory molecular signature. VM-derived ADMSCs share a common \\nproinflammatory molecular signature similar to that induced in 2D ADMSC \\nmonolayers by tumor necrosis factor (TNF)-alpha and are characterized by \\nupregulated expression of COX2, CCL2, CCL5, CXCL5, CXCL8, IL-6, SNAI1, and MMP9. \\nInterestingly, pharmacological inhibition or gene silencing of the JAK2/STAT3 \\nsignaling pathway reduced chemotactic cell migration, in vitro VM and the \\nexpression of proinflammatory and invasive biomarkers.\\nCONCLUSIONS: Overall, we provide novel evidence that inhibiting \\nJAK2/STAT3-regulated VM can also alter the acquisition of a proinflammatory \\nsignature and prevent the contribution of ADMSCs to alternative tumor \\nneovascularization processes.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12964-025-02298-6\\nPMID: 40537799\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: Not applicable. Competing \\ninterests: The authors declare no competing interests.\",\n",
       " '1. J Exp Clin Cancer Res. 2025 Jun 19;44(1):178. doi: 10.1186/s13046-025-03434-3.\\n\\nBaseline levels and dynamic changes of cfDNA, tumor fraction and mutations to \\nanticipate the clinical course of small cell lung cancer (SCLC) patients treated \\nwith first-line atezolizumab and chemotherapy: an hypothesis generating study \\n(CATS/ML43257).\\n\\nPasello G(#)(1)(2), Pigato G(#)(3), Scattolin D(4)(5), Lando S(4)(6), Potente \\nS(7), Romualdi C(7), Roma A(8), Resi MV(9), Frega S(5), Ferro A(5), Maso AD(5), \\nBonanno L(4)(5), Guarneri V(4)(5), Lazzarini E(#)(3), Indraccolo S(#)(4)(3).\\n\\nAuthor information:\\n(1)Department of Surgery, Oncology and Gastroenterology, University of Padova, \\nPadova, Italy. giulia.pasello@unipd.it.\\n(2)Medical Oncology 2, Veneto Institute of Oncology IOV-IRCCS, Padova, Italy. \\ngiulia.pasello@unipd.it.\\n(3)Basic and Translational Oncology Unit, Veneto Institute of Oncology \\nIOV-IRCCS, Padova, Italy.\\n(4)Department of Surgery, Oncology and Gastroenterology, University of Padova, \\nPadova, Italy.\\n(5)Medical Oncology 2, Veneto Institute of Oncology IOV-IRCCS, Padova, Italy.\\n(6)Unit of Biostatistics, Epidemiology and Public Health, Department of Cardiac, \\nThoracic, Vascular Sciences and Public Health, University of Padova, Padova, \\nItaly.\\n(7)Department of Biology, University of Padova, Padova, Italy.\\n(8)Medical Oncology 3, Veneto Institute of Oncology IOV-IRCCS, Castelfranco \\nVeneto, Italy.\\n(9)Medical Oncology AULSS, 8 San Bortolo Hospital, Vicenza, Italy.\\n(#)Contributed equally\\n\\nBACKGROUND: Atezolizumab (A) plus carboplatin-etoposide (CE) represents the new \\nfirst-line treatment in extensive stage (ES)-Small Cell Lung Cancer (SCLC) \\npatients. This study aims at identifying the association of baseline and dynamic \\nchanges of cfDNA, Tumor Fraction (TF) and variant allele frequency (VAF) of \\ntumor-related mutations with median (m) overall (OS) and progression free \\nsurvival (PFS) in SCLC patients treated with ACE.\\nMATERIALS AND METHODS: This is a single-center prospective exploratory study \\nincluding treatment-naive ES-SCLC patients eligible to first-line ACE. Liquid \\nbiopsies were longitudinally collected at baseline (T0), after cycle 1 (T1) and \\n2 (T2), at disease progression (T3). cfDNA Next Generation Sequencing (NGS) \\nanalysis was performed; genomic profiles and TF were inferred from shallow WGS \\n(sWGS).\\nRESULTS: Thirty-two patients were included; mPFS and mOS were 5.19 and 7.96 \\nmonths, respectively. Higher T0 cfDNA (HR 1.44, 95% CI 1.17-1.77, p\\u2009=\\u20090.0006) \\nand VAF (HR 2.6, 95% CI 1.36-4.93, p\\u2009=\\u20090.0039) were associated with risk of \\ndeath; higher T0 cfDNA (HR 1.29, 95% CI 1.08-1.54, p\\u2009=\\u20090.0049), TF (HR 1.97, 95% \\nCI 1.02-3.82, p\\u2009=\\u20090.044) and VAF (HR 2.32, 95% CI 1.22-4.42, p\\u2009=\\u20090.01) were \\npredictors of risk of PD. Among the dynamic changes in the biomarkers under \\ninvestigation, the association of 10-unit increase of VAF T0-T1 and T0-T2 with \\nOS (HR 1.38, 95% CI 1.01-1.88, p\\u2009=\\u20090.043; HR 1.56, 95% CI 1.21-2.16, p\\u2009=\\u20090.008) \\nand PFS (HR 1.69, 95% CI 1.18-2.43, p\\u2009=\\u20090.004; HR 1.81, 95% CI 1.22-2.70, \\np\\u2009=\\u20090.003) was estimated.\\nCONCLUSION: T0 and dynamic changes of cfDNA, TF and VAF may help physicians to \\nstratify ES-SCLC patients receiving first-line ACE and to anticipate the \\nclinical course of the disease.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s13046-025-03434-3\\nPMID: 40537796\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: This study has been approved by Istituto Oncologico Veneto Ethical \\nCommittee (Cod. Int CESC IOV: 2021-85). All patients signed an informed consent \\nform before enrollment. Consent for publication: All patient information has \\nbeen anonymised and consent for publication of the current study was not \\nrequired. All co-authors agreed for publication. Competing interests: The \\nauthors declare no competing interests.',\n",
       " '1. Cell Commun Signal. 2025 Jun 19;23(1):293. doi: 10.1186/s12964-025-02129-8.\\n\\nThe Src family kinase inhibitor drug Dasatinib and glucocorticoids display \\nsynergistic activity against tongue squamous cell carcinoma and reduce MET \\nkinase activity.\\n\\nHmedat ANA(1)(2), Doondeea J(3), Ebner D(4), Feller SM(1), Lewitzky M(5).\\n\\nAuthor information:\\n(1)Institute of Molecular Medicine, Medical Faculty of the \\nMartin-Luther-University Halle-Wittenberg, Halle (Saale), Germany.\\n(2)Department of Pharmaceutics and Pharmaceutical Technology, Faculty of \\nPharmacy, Yarmouk University, Irbid, Jordan.\\n(3)Nuffield Department of Surgical Sciences, Medical Sciences Division, \\nUniversity of Oxford, Oxford, UK.\\n(4)Nuffield Department of Medicine, University of Oxford, Oxford, UK.\\n(5)Institute of Molecular Medicine, Medical Faculty of the \\nMartin-Luther-University Halle-Wittenberg, Halle (Saale), Germany. \\nmarc.lewitzky@uk-halle.de.\\n\\nBACKGROUND: Tongue squamous cell carcinoma (TSCC) is an aggressive cancer \\nassociated with a poor prognosis and limited treatment options, necessitating \\nnew drug targets to improve therapeutic outcomes. Our current work studies \\nprotein tyrosine kinases as well-known targets for successful cancer therapies. \\nIt focuses on Src family kinases (SFK), which are known to play a critical role \\nin some head and neck tumors.\\nMETHODS: Western blot analyses of phospho-tyrosine protein patterns in 34 TSCC \\nlines facilitated the investigation of SFK as contributors to these \\nphosphorylations. The SFK inhibitors PP2 and Dasatinib were utilized to \\ndetermine SFK contributions to cell motility and survival. A high-throughput \\nscreen with 1600 FDA-approved drugs was performed with three TSCC lines to \\ndiscover drugs that act synergistically with Dasatinib against TSCC cell \\nviability. Glucocorticoids emerged as potential candidates and were further \\ninvestigated in 2D culture and by 3D soft agar colony formation. Dexamethasone \\nwas chosen as the major tool for our analyses of synergistic effects of \\nDasatinib and glucocorticoids on TSCC lines. Effects on the cell cycle were \\ninvestigated by flow cytometry and expression levels of cell cycle regulators. \\nSenescence was analyzed by senescence-associated β galactosidase detection and \\np27Kip1 protein expression. Autophagy was measured by Acridine Orange staining.\\nRESULTS: A panel of 34 TSCC lines showed a surprisingly homogenous pTyr-protein \\npattern and a prominent 130 kDa pTyr-protein. Inhibition of SFK activity greatly \\nreduced overall pTyr-protein levels and p130Cas tyrosine phosphorylation. It \\nalso impaired TSCC viability in 2D cell culture and 3D soft agar colony \\nformation. A high-throughput drug combination screen with Dasatinib identified \\nglucocorticoids as promising candidates for synergistic activity. Dasatinib and \\nDexamethasone combination treatment showed strong synergistic effects on Src and \\np130Cas phosphorylation and led to reduced p130Cas expression. Dexamethasone \\nalso suppressed phosphorylation of the MET kinase and its key substrate Gab1. On \\nthe cellular level, Dasatinib combination with glucocorticoids led to G1 cell \\ncycle arrest, appeared to increase senescence and enhanced autophagy. This was \\nalso reflected by effects on cell cycle regulatory proteins, including CDKs and \\ncyclins.\\nCONCLUSION: This work is the first to show a strong synergistic activity of \\nDasatinib in combination with clinically used glucocorticoids in solid tumors. \\nFurthermore, the tyrosine kinase MET and its effector protein Gab1 are newly \\nidentified glucocorticoid targets. Given the extensive research on MET as a drug \\ntarget in various cancers, our findings have the potential to advance future \\ncancer treatments.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12964-025-02129-8\\nPMID: 40537792\\n\\nConflict of interest statement: Declarations. Competing interests: The authors \\ndeclare no competing interests.',\n",
       " \"1. J Neuroinflammation. 2025 Jun 19;22(1):160. doi: 10.1186/s12974-025-03426-2.\\n\\nKChIP3 fosters neuroinflammation and synaptic dysfunction in the 5XFAD mouse \\nmodel of Alzheimer's disease.\\n\\nArcos-Encarnación B(#)(1), Cortes-Flores E(#)(1), Barón-Mendoza I(2), Almazán \\nJL(1), Valle-García D(1)(3), Díaz de León-Guerrero S(1), Hovan L(4), Meza-Sosa \\nKF(1)(5), Camacho-Concha N(1), Gil J(6), Kuijjer ML(4)(7)(8), González-Arenas \\nA(2), Encarnación-Guevara S(9), Pedraza-Alva G(1), Pérez-Martínez L(10).\\n\\nAuthor information:\\n(1)Laboratorio de Neuroinmunobiología, Departamento de Medicina Molecular y \\nBioprocesos, Instituto de Biotecnología, Universidad Nacional Autónoma de México \\n(UNAM), 62210, Cuernavaca, Morelos, México.\\n(2)Departamento de Medicina Genómica y Toxicología Ambiental, Instituto de \\nInvestigaciones Biomédicas, UNAM, 04510, Ciudad de Mexico, México.\\n(3)Laboratorio de Neuroinmunología, Instituto Nacional de Neurología y \\nNeurocirugía Manuel Velasco Suárez (INNNMVS), 14269, Ciudad de Mexico, México.\\n(4)Norwegian Centre for Molecular Biosciences and Medicine (NCMBM), Nordic EMBL \\nPartnership, University of Oslo, Gaustadalléen 21, 0318, Oslo, Norway.\\n(5)Laboratorio de Neurobioquímica y Conducta, Instituto Nacional de Neurología y \\nNeurocirugía Manuel Velasco Suárez (INNNMVS), 14269, Ciudad de Mexico, México.\\n(6)Clinical Chemistry Section, Department of Translational Medicine, Lund \\nUniversity, 22242 Lund, Sweden.\\n(7)iCAN Flagship in Digital Precision Cancer Medicine, University of Helsinki, \\nHaartmanninkatu 8, 00290, Helsinki, Finland.\\n(8)Department of Biochemistry and Developmental Biology, University of Helsinki, \\nHaartmanninkatu 8, 00290, Helsinki, Finland.\\n(9)Laboratorio de Proteómica, Centro de Ciencias Genómicas UNAM, 62210, \\nCuernavaca, Morelos, México.\\n(10)Laboratorio de Neuroinmunobiología, Departamento de Medicina Molecular y \\nBioprocesos, Instituto de Biotecnología, Universidad Nacional Autónoma de México \\n(UNAM), 62210, Cuernavaca, Morelos, México. leonor.perez@ibt.unam.mx.\\n(#)Contributed equally\\n\\nAlzheimer's disease (AD) is a progressive neurodegenerative disorder marked by \\nβ-amyloid (βA) accumulation, neuroinflammation, excessive synaptic pruning, and \\ncognitive decline. Despite extensive research, effective treatments remain \\nelusive. Here, we identify potassium channel-interacting protein 3 (KChIP3) as a \\nkey driver of AD pathology using the 5XFAD mouse model. KChIP3 levels were \\nsignificantly elevated in the hippocampus of 5XFAD mice, correlating with βA \\nburden and neuroinflammation. This upregulation was triggered by inflammatory \\nsignaling via the NLRP3 inflammasome and Caspase-1 activation. Notably, genetic \\ndeletion of KChIP3 (5XFAD/KChIP3-/-) markedly reduced βA plaque deposition, \\npro-inflammatory cytokines, reactive gliosis, and expression of \\ninflammation-related proteins (APO, CLU, MDK). Transcriptomic and proteomic \\nanalyses revealed restored synaptic markers (CD47, CD200, CACNB4, GDA) and a \\nshift of the disease-associated microglial (DAM-1) phenotype. Mechanistically, \\nwe propose that KChIP3 amplifies AD pathology through two key mechanisms: (1) \\nsustaining neuroinflammation by upregulating pro-inflammatory genes and (2) \\nimpairing synaptic integrity by repressing genes critical for neuronal function. \\nConsistently, KChIP3 deletion enhanced dendritic complexity, synaptic \\nplasticity, and cognitive performance in 5XFAD mice. These findings position \\nKChIP3 as a potential therapeutic target for mitigating neuroinflammation and \\nsynaptic dysfunction in AD and highlight its potential as a biomarker for \\ndisease progression.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12974-025-03426-2\\nPMID: 40537787\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: The Institutional Bioethical Committee approved all animal \\nexperiments described in this study. Competing interests: The authors declare no \\ncompeting interests.\",\n",
       " \"1. J Hematol Oncol. 2025 Jun 19;18(1):65. doi: 10.1186/s13045-025-01717-y.\\n\\nExtracellular matrix dynamics in tumor immunoregulation: from tumor \\nmicroenvironment to immunotherapy.\\n\\nHu Q(1), Zhu Y(1), Mei J(2), Liu Y(3), Zhou G(4).\\n\\nAuthor information:\\n(1)The Affiliated Cancer Hospital of Nanjing Medical University, Jiangsu Cancer \\nHospital, Jiangsu Institute of Cancer Research, Nanjing, Jiangsu, People's \\nRepublic of China.\\n(2)The First Clinical Medicine College, Nanjing Medical University, Nanjing, \\nJiangsu, People's Republic of China.\\n(3)Jiangsu Cancer Hospital, The Affiliated Cancer Hospital of Nanjing Medical \\nUniversity, Jiangsu Institute of Cancer Research, Nanjing, Jiangsu, People's \\nRepublic of China. liuying5250@163.com.\\n(4)The Affiliated Cancer Hospital of Nanjing Medical University, Jiangsu Cancer \\nHospital, Jiangsu Institute of Cancer Research, Nanjing, Jiangsu, People's \\nRepublic of China. zhouguoren888@126.com.\\n\\nThe extracellular matrix (ECM), closely linked to the dynamic changes in the \\ntumor microenvironment (TME), plays a critical role in modulating tumor \\nimmunity. The dual role of the ECM in tumor progression, encompassing both \\npromotion and inhibition, is attributed to its components influencing immune \\ncell activation, migration, and infiltration. This mechanism is intricately \\nconnected with the efficacy of immunotherapies. Currently, there is limited \\nunderstanding of how ECM remodeling spatially and temporally coordinates with \\nimmune checkpoint inhibitors (ICIs) or adoptive cell therapies. Furthermore, \\nstrategies to selectively target pathological ECM components while preserving \\ntheir homeostatic functions urgently require systematic investigation. In this \\nreview, we summarize current findings on the interplay between ECM and tumor \\nimmune regulation, with a particular focus on how key ECM components contribute \\nto immune modulation. Furthermore, we discuss emerging strategies targeting \\nECM-related mechanisms to enhance the efficacy of immunotherapies, including \\napproaches that remodel the ECM to improve immune infiltration and strategies \\nthat synergize with existing immunotherapies. By integrating these insights, we \\nprovide a perspective on leveraging ECM-targeted interventions to overcome \\nimmune evasion and optimize cancer immunotherapy outcomes.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s13045-025-01717-y\\nPMID: 40537775\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: This manuscript has been \\nread and approved by all the authors to publish and is not submitted or under \\nconsideration for publication elsewhere. Competing interests: The authors \\ndeclare no competing interests.\",\n",
       " '1. BMC Biol. 2025 Jun 19;23(1):166. doi: 10.1186/s12915-025-02254-3.\\n\\nThe phosphatase PPM1F, a negative regulator of integrin activity, is essential \\nfor embryonic development and controls tumor cell invasion.\\n\\nGrimm TM(#)(1)(2), Dierdorf NI(#)(1)(2), Herbinger M(#)(1), Baumgärtner S(1)(2), \\nSontowski E(1), Paone C(1)(2), Baade T(1)(2), Hauck CR(3)(4).\\n\\nAuthor information:\\n(1)Lehrstuhl Zellbiologie, Fachbereich Biologie, Maildrop 621, Universität \\nKonstanz, Universitätsstrasse 10, Konstanz, Germany.\\n(2)Konstanz Research School Chemical Biology, Universität Konstanz, Konstanz, \\nGermany.\\n(3)Lehrstuhl Zellbiologie, Fachbereich Biologie, Maildrop 621, Universität \\nKonstanz, Universitätsstrasse 10, Konstanz, Germany. \\nchristof.hauck@uni-konstanz.de.\\n(4)Konstanz Research School Chemical Biology, Universität Konstanz, Konstanz, \\nGermany. christof.hauck@uni-konstanz.de.\\n(#)Contributed equally\\n\\nBACKGROUND: The Mn2+/Mg2+-dependent Ser/Thr phosphatase PPM1F was identified to \\ncontrol integrin activity. Furthermore, PPM1F regulates several protein kinases \\nknown to be involved in organizing the cytoskeleton and other cellular \\nfunctions. Therefore, PPM1F appears critical for a multitude of physiological \\nprocesses.\\nRESULTS: Here, we report the phenotype of ppm1f gene disruption in mice. While \\nheterozygous ppm1f\\u2009±\\u2009mice are viable and fertile, ppm1f-/- mice show severe \\ndefects and significant morphological abnormalities in the developing brain and \\nvasculature and abort embryonic development at day E10.5. Isolated ppm1f-/- MEFs \\nor PPM1F-depleted human neuro-epithelial cells display enhanced \\nintegrin-dependent cell adhesion, deregulated PAK phosphorylation, and perturbed \\ncell migration. These phenotypes were reversed by re-expression of the wildtype \\nenzyme, but not the phosphatase-inactive PPM1F. In different human tumor cell \\ntypes, PPM1F expression levels directly correlated with invasive potential, \\nwhile deletion of PPM1F abrogates tissue invasion.\\nCONCLUSIONS: These results highlight the non-redundant role of this enzyme in \\nintegrin and PAK regulation and identify PPM1F as a promising target to limit \\ntumor metastasis.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12915-025-02254-3\\nPMID: 40537771\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Mice were kept in accordance with relevant institutional and \\nnational guidelines and regulations in the central animal care facility of \\nUniversity of Konstanz. Procurement of mouse tissues was in accordance with the \\nanimal welfare guidelines 2010/63/EC of the European Communities Council \\nDirective and was registered with official authorities (Regierungspräsidium \\nFreiburg, Germany). Consent for publication: Not applicable. Competing \\ninterests: The authors declare having no competing interests.',\n",
       " '1. Cancer Cell Int. 2025 Jun 19;25(1):217. doi: 10.1186/s12935-025-03868-x.\\n\\nLncRNA H19 acts as a ceRNA to promote glioblastoma malignancy by sponging \\nmiR-19b-3p and upregulating SERPINE1.\\n\\nJang W(1), Im M(1), Yoon G(1), Roh J(2), Kim W(3)(4).\\n\\nAuthor information:\\n(1)Department of Science Education, Korea National University of Education, \\nCheongju-si, 28173, Chungbuk, Republic of Korea.\\n(2)Department of Biology Education, Seowon University, Cheongju-si, 28674, \\nChungbuk, Republic of Korea.\\n(3)Department of Science Education, Korea National University of Education, \\nCheongju-si, 28173, Chungbuk, Republic of Korea. wykim82@knue.ac.kr.\\n(4)Department of Biology Education, Korea National University of Education, \\nCheongju-si, 28173, Chungbuk, Republic of Korea. wykim82@knue.ac.kr.\\n\\nBACKGROUND: Glioblastoma (GBM) is a highly aggressive brain tumor, characterized \\nby genetic complexity and resistance to treatment. Despite significant \\nadvancements in cancer research, the mechanisms driving GBM progression remain \\npoorly understood. This study investigates the molecular pathways associated \\nwith GBM, and focuses on long non-coding RNA H19 and its role in tumor growth \\nand progression.\\nMETHODS: Analysis of Gene Expression Omnibus (GEO) dataset was performed to \\nidentify differentially expressed genes in GBM cells with H19 overexpression, \\nrevealing serpin family E member 1 (SERPINE1) as a potential target. \\nBioinformatics analyses were used to evaluate the differential expression of H19 \\nand SERPINE1 in GBM tissues, perform survival analysis, and predict miR-19b-3p \\nas a candidate miRNA. The expression levels of H19, miR-19b-3p, and SERPINE1 \\nwere validated using RT-qPCR and Western blotting. Dual-luciferase reporter \\nassays were conducted to confirm the direct interactions between H19, \\nmiR-19b-3p, and SERPINE1. Cell viability and motility assays were performed to \\nassess the effects of modulating H19/miR-19b-3p/SERPINE1 expression on cell \\nsurvival, migration, and invasion.\\nRESULTS: Bioinformatics analyses identified SERPINE1 as an oncogene upregulated \\nin GBM cells with H19 overexpression, and the overexpression of H19 and SERPINE1 \\nwas linked to poor prognosis in GBM patients. Experimental validation \\ndemonstrated that H19 upregulates SERPINE1 expression, while miR-19b-3p directly \\nbinds to both H19 and SERPINE1, suppressing SERPINE1 expression. Functional \\nassays further confirmed that H19 promotes cell survival, migration, and \\ninvasion, whereas miR-19b-3p inhibits these processes by downregulating \\nSERPINE1.\\nCONCLUSIONS: These findings reveal a novel mechanism whereby H19 drives GBM \\nprogression by acting as a competing endogenous RNA (ceRNA) to sponge miR-19b-3p \\nand upregulate SERPINE1 expression. Our results offer new insights into the \\nregulatory interplay among H19, miR-19b-3p, and SERPINE1 in GBM cell lines, a \\nrelationship that has not been clearly defined in previous research. Moreover, \\nthe H19/miR-19b-3p/SERPINE1 axis highlights the potential use of H19, \\nmiR-19b-3p, and SERPINE1 as promising biomarkers and therapeutic targets in GBM.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12935-025-03868-x\\nPMID: 40537770\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: Not applicable. Competing \\ninterests: The authors declare no competing interests.',\n",
       " '1. J Transl Med. 2025 Jun 19;23(1):689. doi: 10.1186/s12967-025-06732-8.\\n\\nFunctions and mechanisms of lncRNAs in immune escape and their application in \\nimmunotherapy for colorectal cancer.\\n\\nZhang D(#)(1), Pei S(#)(1), Feng Z(#)(1), Xia G(2).\\n\\nAuthor information:\\n(1)Queen Mary School, Jiangxi Medical College, Nanchang University, Nanchang, \\n330031, China.\\n(2)Department of Hepatobiliary Surgery, Jiujiang University Affiliated Hospital, \\nJiujiang, 332000, China. 18879254018@163.com.\\n(#)Contributed equally\\n\\nRecent studies have progressively clarified the critical role of long non-coding \\nRNA (lncRNA) in cancer, especially concerning its involvement in immune \\nprocesses, which holds significant therapeutic promise. However, the \\nincorporation of lncRNA into immunotherapy has yet to demonstrate adequate \\nclinical efficacy. This shortcoming can be attributed to various challenges, \\nincluding unclear mechanisms of action and limited prognostic capacity. In \\ncolorectal cancer (CRC), this ineffectiveness may predominantly stem from immune \\nescape mechanisms mediated by lncRNAs. Despite some exploration into \\nlncRNA-driven immune escape, a comprehensive overview of its implications for \\nimmunotherapy in CRC remains inadequately addressed. In this review, we \\nthoroughly assess the role of lncRNA in immune escape mechanisms associated with \\nCRC, focusing on immune checkpoints, immune cells, and cytokines. We elucidate \\nthe relationship between lncRNA-mediated immune escape and CRC progression and \\ndiscuss strategies for lncRNA-mediated immunotherapy for CRC. Our aim is to \\nprovide fresh insights into the potential of targeting lncRNAs to enhance the \\neffectiveness of immunotherapeutic strategies in CRC.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12967-025-06732-8\\nPMID: 40537762\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: Not applicable. Competing \\ninterests: The authors declare no competing interests.',\n",
       " '1. BMC Biotechnol. 2025 Jun 19;25(1):48. doi: 10.1186/s12896-025-00991-5.\\n\\nExploring the mechanisms and mode of action of bioactive compounds from marine \\nStreptomyces albidoflavus against breast carcinoma cells.\\n\\nShata AM(#)(1), Rejili MS(2), El-Naggar MM(3), El-Asser SA(4), Saleh \\nAA(#)(5)(6), Hafez EE(7), Haouala F(2), Youssef AS(4).\\n\\nAuthor information:\\n(1)National Institute of Oceanography and Fisheries, NIOF, Cairo, Egypt.\\n(2)Department of Biology, College of Sciences, Imam Mohammad Ibn Saud Islamic \\nUniversity (IMSIU), Riyadh, 11623, Saudi Arabia.\\n(3)National Institute of Oceanography and Fisheries, NIOF, Cairo, Egypt. \\nmmelnaggar@yahoo.com.\\n(4)Faculty of Science, Department of Botany and Microbiology, Alexandria \\nUniversity, Alexandria, Egypt.\\n(5)College of Animal Science & Technology, Yangzhou University, Yangzhou, \\nJiangsu, 225009, China. Elemlak1339@gmail.com.\\n(6)Faculty of Agriculture (Al-Shatby), Animal and Fish Production Department, \\nAlexandria University, Alexandria City, 11865, Egypt. Elemlak1339@gmail.com.\\n(7)Department of Plant Protection and Biomolecular Diagnosis, ALCRI City of \\nScientific Research and Technological Applications, Alexandria, Egypt.\\n(#)Contributed equally\\n\\nBreast cancer remains one of the most lethal diseases for women worldwide. \\nMarine populations are considered a vast reservoir for novel bioactive \\nmetabolites, particularly marine Actinomycetes, which are known to produce \\nvarious bioactive compounds with antitumour, antibacterial, and antifungal \\nproperties. A promising new marine strain was isolated and identified as \\nStreptomyces albidoflavus strain EgyAB2 (16\\xa0S rRNA gene sequence accession \\nnumber ON680945.1). The anticancer activity of the extracted compounds was \\ntested in the MCF7 cell line using a sulforhodamine B (SRB) bioassay, which \\nrevealed an IC50 of 0.36\\xa0µg/ml compared to the chemotherapeutic drug \\n5-fluorouracil (0.35\\xa0µg/ml). Additionally, the anticancer activity was confirmed \\nby a dimethyl-thiazol-diphenyl-tetrazolium bromide (MTT) bioassay, which showed \\nan IC50 of 17.46\\xa0µg/ml. The mode of action of the treated breast carcinoma \\n(apoptotic effect) was studied via qRT-PCR, revealing a significant role in \\nanticancer treatment. Although the extracted compounds exhibited high \\nantioxidant activity in the diphenyl-2-picrylhydrazyl (DPPH) radical-scavenging \\nassay, they presented an IC50 of 8.92\\xa0µg/ml and an inhibition percentage of \\n56.08%. Chemical characterisation was performed via GC‒MS, 1\\xa0H-NMR, and FTIR \\nspectroscopy analyses, revealing the presence of 2-D N-methyl imidazole, \\n2-nonadecene, 1-D-2-methyl imidazole, and propane dinitrile, all of which \\nexhibit antitumour activity.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12896-025-00991-5\\nPMID: 40537753\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: The study was applicable and approved by the ethics committee of \\nthe Faculty of Medicine, Alexandria University, with serial number 0306273. All \\nthe bacterial pathogens used in this study were obtained from the Egyptian \\nCulture Collection Network. Consent for publication: Not applicable. Competing \\ninterests: The authors declare no competing interests.',\n",
       " \"1. BMC Cancer. 2025 Jun 19;25(1):1020. doi: 10.1186/s12885-025-14434-z.\\n\\nThe MAP kinase negative regulator DUSP2 (dual specificity phosphatase 2) is \\ncontrolled by oncogenic microRNA cluster miR-17-92, miR-106a-363 and \\nmiR-106b-25.\\n\\nTenhaken V(1), Seternes OM(2), Cascorbi I(1), Bruckmueller H(3)(4).\\n\\nAuthor information:\\n(1)Institute of Experimental and Clinical Pharmacology, University Hospital \\nSchleswig-Holstein, Campus Kiel, Arnold-Heller-Str. 3, 24105, Kiel, Germany.\\n(2)Department of Pharmacy, UiT The Arctic University of Norway, Tromsø, 9037, \\nNorway.\\n(3)Institute of Experimental and Clinical Pharmacology, University Hospital \\nSchleswig-Holstein, Campus Kiel, Arnold-Heller-Str. 3, 24105, Kiel, Germany. \\nh.bruckmueller@pharmakologie.uni-kiel.de.\\n(4)Department of Pharmacy, UiT The Arctic University of Norway, Tromsø, 9037, \\nNorway. h.bruckmueller@pharmakologie.uni-kiel.de.\\n\\nBACKGROUND: Aberrant changes in protein phosphorylation are a hallmark of \\ncancer, often leading to hyperactivation of signalling pathways such as the \\nmitogen activated protein kinase (MAPK) pathway. Although kinase inhibitors are \\nsuccessfully used in certain clinical indications, drug resistance remains a \\nchallenge, and alternative approaches to control phosphorylation-dependent \\noncogenic signalling are increasingly being considered. These include the \\nmodulation of negative regulators of oncogenic signalling pathways. The \\ndual-specificity phosphatase 2 (DUSP2) is one of the essential negative \\nregulators for the MAPK pathway, providing tight and efficient control of MAPKs \\nunder physiological conditions. However, in oncogenic contexts, negative \\nfeedback regulation is often impaired and the mechanisms controlling DUSP2 \\nexpression and function remain largely elusive. The aim of the present study was \\nto investigate whether microRNA-mediated regulation of DUSP2 could contribute to \\nan impairment of negative feedback regulation in cancer.\\nMETHODS: A combination of in silico target prediction, integrative analysis of \\npan-cancer microRNA and DUSP2 mRNA expression data as well as a literature \\nsearch was applied to identify microRNAs potentially regulating DUSP2 expression \\nin cancer context. Predicted interactions of microRNAs with the DUSP2 3'UTR were \\nverified using reporter gene assays and functionally validated in a lymphoma \\ncell model.\\nRESULTS: A comprehensive analysis of microRNA and DUSP2 mRNA expression data \\nacross 32 cancer types revealed significant inverse correlations between \\noncogenic microRNA clusters (miR-17-92, miR-106a-363, and miR-106b-25 cluster) \\nand DUSP2 expression in various cancer types. Reporter gene assay analysis \\nconfirmed the interaction of miR-17-5p, miR-20a-5p, miR-20b-5p, miR-29b-3p, \\nmiR-93-5p, miR-106b-5p, miR-122-5p, miR-340-5p, miR-520a-3p, and miR-520c-3p \\nwith the DUSP2 mRNA 3'UTR. Furthermore, treatment of the lymphoma cell line \\nWSU-DLCL2 with microRNA inhibitors for miR-17-5p, miR-20b-5p, or miR-106b-5p \\nresulted in increased DUSP2 mRNA levels.\\nCONCLUSION: The results of this study indicate that microRNA-mediated regulation \\nof DUSP2 in hematologic and solid cancers appears to be a plausible mechanism \\nthat contributes to the dysregulation of MAP kinase signaling pathways in cancer \\nby impairing negative feedback regulation. The data provide a solid foundation \\nfor future studies to investigate the consequences of regulation of DUSP \\nfunction in cancer in more depth.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12885-025-14434-z\\nPMID: 40537745\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: Not applicable. Competing \\ninterests: The authors declare no competing interests.\",\n",
       " \"1. BMC Cancer. 2025 Jun 19;25(1):1021. doi: 10.1186/s12885-025-14442-z.\\n\\nAmniotic membrane promotes doxorubicin potency by suppressing SH-SY5Y \\nneuroblastoma cell angiogenesis.\\n\\nAbou-Shanab AM(#)(1)(2), Shouman S(#)(1)(2), Hussein AE(1)(2), Gaser OA(1)(2), \\nMagdy S(1)(2), Ashraf E(1)(2), Salah RA(1)(2), Idris O(3), El-Badri N(4)(5).\\n\\nAuthor information:\\n(1)Center of Excellence for Stem Cells and Regenerative Medicine, Zewail City of \\nScience and Technology, Giza, 12578, Egypt.\\n(2)Biomedical Sciences Program, University of Science and Technology, Zewail \\nCity of Science and Technology, Giza, 12578, Egypt.\\n(3)Department of Obstetrics and Gynecology, Kasr Al-Ainy University Hospital, \\nCairo University, Cairo, Egypt.\\n(4)Center of Excellence for Stem Cells and Regenerative Medicine, Zewail City of \\nScience and Technology, Giza, 12578, Egypt. nelbadri@zewailcity.edu.eg.\\n(5)Biomedical Sciences Program, University of Science and Technology, Zewail \\nCity of Science and Technology, Giza, 12578, Egypt. nelbadri@zewailcity.edu.eg.\\n(#)Contributed equally\\n\\nBACKGROUND: Doxorubicin (DOX) remains a mainstay for neuroblastoma (NB) \\ntreatment, but side effects hamper efficacy. We previously showed that DOX \\ninduces SH-SY5Y NB cell angiogenesis via the PHD-2/HIF-1α axis. Adjuvant \\ntherapies offer a promising avenue to improved outcomes. Human amniotic membrane \\n(hAM) extract (hAME) consists of various proteins that exhibit anti-cancer and \\nanti-angiogenic properties. This study investigates hAME as a potential adjuvant \\nfor targeting NB angiogenesis when combined with DOX.\\nMETHODS: We used cellular, molecular, and biochemical assays to evaluate the \\nantitumorigenic activities of hAME\\u2009+\\u2009DOX (D\\u2009+\\u2009E) treatment across key hallmarks \\nof SH-SY5Y NB progression: proliferation, cell cycle, angiogenesis, \\ninvasiveness, differentiation, and cellular bioenergetics.\\nRESULTS: D\\u2009+\\u2009E treatment significantly suppressed SH-SY5Y cell proliferation, \\ninduced cell cycle perturbations, and reduced viability, while protecting bone \\nmarrow stem cells and human skin fibroblast normal cells. D\\u2009+\\u2009E treatment also \\ncountered SH-SY5Y cell invasiveness and promoted a favorable \\nmesenchymal-to-epithelial transition (MET). Importantly, D\\u2009+\\u2009E treatment \\nmodulated the SH-SY5Y cellular respiration, evidenced by halted glycolytic \\nmetabolism, potentially influencing a shift towards oxidative phosphorylation \\nand boosted urea cycle progression. Mechanistically, D\\u2009+\\u2009E abrogated DOX's \\npro-angiogenic effects and inhibited SH-SY5Y cells' neo-vascularization in a \\nchick embryo model.\\nCONCLUSIONS: These findings suggest hAME as a promising adjuvant therapy for NB, \\npotentially offering an effective and safe treatment strategy by targeting \\nmultiple hallmarks of NB.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1186/s12885-025-14442-z\\nPMID: 40537738\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Human amniotic membrane (hAM) was ethically sourced to ensure \\npatient safety and regulatory compliance. We obtained hAM following elective \\ncesarean sections from healthy subjects who underwent thorough screening and \\ntesting negative for blood-borne infections. Prior to tissue collection, written \\ninformed consent was obtained from all participating subjects. All study \\nprocedures, including patient recruitment, consent acquisition, and hAM \\nprocessing, were reviewed and approved by the Institutional Review Board (IRB) \\nat Sheikh Zayed Specialized Hospital. We adhered to all relevant ethical \\nguidelines and regulations throughout the study to ensure the well-being of the \\nmothers and the integrity of the research. Consent for publication: Not \\napplicable. Competing interests: The authors declare no competing interests.\",\n",
       " '1. Reprod Sci. 2025 Jun 19. doi: 10.1007/s43032-025-01916-5. Online ahead of\\nprint.\\n\\nRole of Defense/Immunity Proteins in Non-Obstructive Azoospermia: Insights from \\nGene Expression and Single-Cell RNA Sequencing Analyses.\\n\\nAbroudi AS(1), Azizi H(2)(3), Djamali M(4), Abdullah HK(5), Qorbanee A(5), \\nSkutella T(6).\\n\\nAuthor information:\\n(1)Department of Cellular and Molecular Biology, Faculty of Advanced Science and \\nTechnology, Tehran Medical Sciences, Islamic Azad University, Tehran, Iran.\\n(2)Faculty of Biotechnology, Amol University of Special Modern Technologies, \\nAmol, Iran. H.azizi@ausmt.ac.ir.\\n(3)Department of Stem Cells and Cancer, College of Biotechnology, Amol \\nUniversity of Special Modern Technologies, Amol, Iran. H.azizi@ausmt.ac.ir.\\n(4)Department of Biology, Faculty of Science, Tehran University, Tehran, Iran.\\n(5)Department of Surgery, Faculty of General of Medicine, Koya University, \\nKurdistan Region - F.R, Koya, KOY45, Iraq.\\n(6)Institute for Anatomy and Cell Biology, Medical Faculty, University of \\nHeidelberg, Im Neuenheimer Feld 307, 69120, Heidelberg, Germany.\\n\\nNon-obstructive azoospermia (NOA) is a severe form of male infertility \\ncharacterized by a complete absence of sperm in the ejaculate due to impaired \\nspermatogenesis. While genetic and hormonal factors are known contributors, \\nrecent evidence highlights the role of immune dysregulation in NOA \\npathophysiology. Defense/immunity proteins play a crucial role in maintaining \\ntesticular immune homeostasis, but their aberrant expression may contribute to \\nspermatogenic failure. This study analyzed gene expression data from multiple \\ndatasets to identify immune-related genes significantly altered in NOA, \\nincluding IFITM1, CES1, MR1, LSAMP, PRB2, CARD8, and PECAM1. Differential \\nexpression analysis revealed upregulation of IFITM1 and PECAM1 and \\ndownregulation of CES1, MR1, LSAMP, PRB2, and CARD8, suggesting immune imbalance \\nand oxidative stress contribute to NOA pathogenesis. Additionally, single-cell \\nRNA sequencing confirmed their differential expression in distinct testicular \\ncell populations. Our findings suggest that immune-related pathways, including \\ncytokine signaling, antigen processing, and oxidative stress response, may \\ncontribute to NOA pathogenesis. Targeting these molecular pathways could provide \\nnovel therapeutic strategies for improving reproductive outcomes in affected \\nindividuals.\\n\\n© 2025. The Author(s), under exclusive licence to Society for Reproductive \\nInvestigation.\\n\\nDOI: 10.1007/s43032-025-01916-5\\nPMID: 40537735\\n\\nConflict of interest statement: Declarations. Institutional Review Board \\nStatement: The Amol University of Special Modern Technologies Ethics Committee \\n(reference number Ir.asmt.rec.1403.125) gave its approval for the research to \\nproceed, and its protocols were followed. Informed Consent Statement: Not \\napplicable. Conflicts of interest: It is declared by the remaining authors that \\nthere are no commercial or financial relationships that might conflict with this \\nresearch.',\n",
       " \"1. Discov Oncol. 2025 Jun 19;16(1):1158. doi: 10.1007/s12672-025-02983-5.\\n\\nPTX3 as a diagnostic and prognostic biomarker in lung adenocarcinoma: a \\ncomprehensive analysis.\\n\\nZhou S(#)(1), Li N(#)(1), Haishaer D(2), Zhao H(3).\\n\\nAuthor information:\\n(1)Department of Clinical Laboratory, China Medical University People's \\nHospital, The People's Hospital of Liaoning Province, Shenyang, China.\\n(2)Department of Clinical Laboratory, The First Affiliated Hospital of Fujian \\nMedical University, Fuzhou, China.\\n(3)Hospital-Acquired Infection Control Department, China Medical University \\nPeople's Hospital, The People's Hospital of Liaoning Province, Shenyang, China. \\n13889299493@126.com.\\n(#)Contributed equally\\n\\nBACKGROUND: Lung cancer remains a leading global cause of mortality, with lung \\nadenocarcinoma (LUAD) as the predominant histological subtype. Current serum \\nbiomarkers like carcinoembryonic antigen (CEA) lack specificity, necessitating \\nnovel diagnostic targets. Pentraxin 3 (PTX3), a homo-multimeric protein \\ndownregulated in malignancies, was evaluated for its diagnostic and prognostic \\nroles in LUAD.\\nMETHODS: PTX3 expression was analyzed using TCGA/GEO datasets and clinical serum \\nsamples (97 LUAD vs. 40 controls). Diagnostic utility was assessed via ROC \\ncurves for PTX3, CEACAM5, and their combination. Prognostic value was determined \\nby Kaplan-Meier and Cox regression. PTX3-associated differentially expressed \\ngenes (DEGs) were explored through functional enrichment, tumor microenvironment \\n(TME) analysis, and drug sensitivity profiling.\\nRESULT: The TCGA and GEO datasets revealed that PTX3 mRNA expression was \\nsignificantly downregulated in LUAD, and the AUC values with PTX3 were >\\u20090.7. \\nDetection of CEACAM5 and PTX3 combined can improve diagnostic accuracy, and \\npatients with high PTX3 level have shorter overall survival. Multivariate Cox \\nanalysis revealed that PTX3 is an independent predictor of overall survival. The \\nresult of ELISA further confirmed the low level of PTX3 protein. PTX3 is \\nimportant in the functional analysis and TME of lung adenocarcinoma. In \\naddition, the sensitivity of tumor cells to anti-cancer drugs is significantly \\ncorrelated with the expression of PTX3.\\nCONCLUSION: PTX3 emerges as a dual biomarker for LUAD diagnosis and prognosis, \\nwith mechanistic ties to TME remodeling and therapeutic resistance, highlighting \\nits potential for clinical translation.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s12672-025-02983-5\\nPMID: 40537732\\n\\nConflict of interest statement: Declarations. Competing interests: The authors \\ndeclare no competing interests. Ethics approval and consent to participate: This \\nstudy was performed in line with the principles of the Declaration of Helsinki. \\nThe Ethics Committee of China Medical University People’s Hospital evaluated and \\napproved this work (approval number 2024-K069). Informed consent was obtained \\nfrom all individual participants included in the study.\",\n",
       " '1. Clin Transl Oncol. 2025 Jun 19. doi: 10.1007/s12094-025-03961-7. Online ahead\\nof  print.\\n\\nCan intraoperative sentinel node fine-needle aspiration biopsy replace sentinel \\nnode biopsy in early-stage breast cancer patients undergoing breast-conserving \\nsurgery?\\n\\nBarco I(1), García A(2), Vallejo E(2), Torras M(2), Tarroch X(2), Ysamat M(2), \\nFraile M(2).\\n\\nAuthor information:\\n(1)Breast Unit, Department of Gynecology, Hospital Universitari Mútua Terrassa, \\nUniversitat de Barcelona, Terrassa, Spain. ibarco@mutuaterrassa.es.\\n(2)Breast Unit, Department of Gynecology, Hospital Universitari Mútua Terrassa, \\nUniversitat de Barcelona, Terrassa, Spain.\\n\\nBACKGROUND: Sentinel Node Biopsy (SNB) side effects could be avoided if the \\nsentinel node (SN) is sampled but not removed using intraoperative fine-needle \\naspiration biopsy (FNAB). In a previous study, we demonstrated the feasibility \\nof this technique. The goal of our current research is to evaluate its potential \\nas an alternative to SNB in node-negative patients undergoing breast-conserving \\nsurgery.\\nPATIENTS AND METHODS: We studied 153 cases referred to our Breast Unit for \\ntreatment involving conservative breast surgery and SNB. Patients receiving \\nneoadjuvant chemotherapy were excluded. We assessed the sensitivity, \\nspecificity, and negative predictive value (NPV) of intraoperative SN-FNAB by \\ncomparing it with histopathology results and the need for subsequent axillary \\nlymph node dissection (ALND) across three levels of analysis: direct pathologic \\ncomparison, exclusion of micrometastases, and exclusion of up to two involved \\nsentinel nodes.\\nRESULTS: In 81.7% of cases, samples were suitable for cytologic analysis. Both \\nsensitivity and NPV reached 100% in the third level of analysis. No negative \\ncases showed tumor involvement beyond the sentinel node. Positive FNAB results \\nwere associated with a higher likelihood of ALND and greater axillary tumor \\nburden.\\n\\n© 2025. The Author(s), under exclusive licence to Federación de Sociedades \\nEspañolas de Oncología (FESEO).\\n\\nDOI: 10.1007/s12094-025-03961-7\\nPMID: 40537727\\n\\nConflict of interest statement: Declarations. Conflict of interest: The authors \\ndeclare no competing financial interests or personal relationships that could \\ninfluence the work reported in this study. Human participants and/or animals and \\nInformed consent: Every patient has been informed and the inform consent has \\nbeen signed.',\n",
       " '1. Clin Transl Oncol. 2025 Jun 19. doi: 10.1007/s12094-025-03972-4. Online ahead\\nof  print.\\n\\nDevelopment of a practice-based score to predict extended duration of proton \\nbeam therapy session in pediatric patients.\\n\\nFernández-de Miguel JM(1), García-Aroca MÁ(2), Manrique Yera I(3), Calvo FA(4), \\nAymerich de Francesci M(1), Álvarez Avello JM(1), Panizo Morgado E(5), \\nNúñez-Córdoba JM(6)(7)(8).\\n\\nAuthor information:\\n(1)Department of Anaesthesia and Critical Care, Clínica Universidad de Navarra, \\nMadrid, Spain.\\n(2)Department of Anesthesia and Intensive Care, HCD \"Gómez-Ulla\", Madrid, Spain.\\n(3)Operations Department and Protontherapy Unit, Clínica Universidad de Navarra, \\nMadrid, Spain.\\n(4)Department of Radiation Oncology, Cancer Center, Clinica Universidad de \\nNavarra, Madrid, Spain.\\n(5)Department of Pediatric Oncology, Clínica Universidad de Navarra, Madrid, \\nSpain.\\n(6)Research Support Service, Central Clinical Trials Unit, Clínica Universidad \\nde Navarra, Pio XII, 36, 31008, Pamplona, Spain. jnunezco@unav.es.\\n(7)Institute of Data Science and Artificial Intelligence, University of Navarra, \\n31009, Pamplona, Spain. jnunezco@unav.es.\\n(8)Department of Health Sciences, Public University of Navarra, 31008, Pamplona, \\nSpain. jnunezco@unav.es.\\n\\nPURPOSE: Due to the labor intensity demanded by proton beam therapy (PBT) in \\npediatric patients, information on operational procedures related to efficiency \\nis crucial to optimize quality and safety. We aimed to identify patient factors \\nthat affect the duration of the pediatric PBT session and to develop an \\neasy-to-use predictive score of extended duration.\\nMETHODS/PATIENTS: This is an observational retrospective cohort study in an \\nacademic medical centre, between May 2020 and February 2024. Seventy seven ASA \\nIII pediatric patients treated with PBT were recruited.\\nRESULTS: The mean age was 4.8\\xa0years [standard deviation (SD): 2.1] and 52% were \\nwomen. The mean duration of the PBT session was 50\\xa0min (SD: 17). Extended \\nduration of the PBT session (>\\u200945\\xa0min) occurred in 39 patients (51%). Five \\npredictors of extended duration were selected for the final prediction model. In \\nthe multivariable model, an age\\u2009>\\u200945\\xa0months showed a near eightfold increased \\nodds of extended duration [Odds ratio (OR): 7.76, 95% confidence interval (95% \\nCI) 1.63-36.99, P\\u2009=\\u20090.010]. The OR (95% CI) for long-term venous access, no \\nrecurrent tumors, hydrocephalus, and craniospinal location were 5.91 (1.47 to \\n23.79), 3.81 (0.67 to 21.69), 3.79 (0.90 to 15.97), and 2.59 (0.69 to 9.76), \\nrespectively. This five-variable model was used to build a nomogram-based score \\nwith an area under the receiver operating characteristic curve of 0.84 (95% CI \\n0.76-0.93).\\nCONCLUSIONS: A simple nomogram based on readily available pretreatment data has \\npotential for planning pediatric PBT standard clinical expert practice.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s12094-025-03972-4\\nPMID: 40537726\\n\\nConflict of interest statement: Declarations. Conflict of interest: The authors \\nhave no competing interests to declare that are relevant to the content of this \\narticle. Ethical approval and informed consent: The study was approved by the \\nResearch Ethics Committee of the University of Navarra (Date: June 13, 2024. \\nProject number: 2024.11) and adhered to the Revised Declaration of Helsinki \\n[13]. The need to obtain informed written consent from patients was waived due \\nto the retrospective nature of the study.',\n",
       " '1. Clin Transl Oncol. 2025 Jun 19. doi: 10.1007/s12094-025-03967-1. Online ahead\\nof  print.\\n\\nCausal inference in the diagnosis and prognosis of ovarian cancer: current state \\nand future directions.\\n\\nZhan F(#)(1)(2), He L(#)(3), Qin S(1), Guo Y(4).\\n\\nAuthor information:\\n(1)School of Electronic Information Engineering, Taiyuan University of Science \\nand Technology, Taiyuan, 030024, Shanxi, China.\\n(2)College of Engineering, Fujian Jiangxia University, Fuzhou, 350108, Fujian, \\nChina.\\n(3)Department of Obstetrics and Gynecology, The First Affiliated Hospital of \\nFujian Medical University, Fuzhou, 350004, Fujian, China.\\n(4)School of Electronic Information Engineering, Taiyuan University of Science \\nand Technology, Taiyuan, 030024, Shanxi, China. zulibest@tyust.edu.cn.\\n(#)Contributed equally\\n\\nOvarian cancer represents one of the most lethal gynecologic malignancies, \\ncharacterized by low early detection rates and challenging prognostic \\nassessment. Conventional diagnostic modalities demonstrate limited sensitivity \\nand specificity for early-stage disease identification. Recent research has \\nbegun to explore causal inference methodologies as complementary approaches that \\nmay enhance diagnostic precision and prognostic capability. This systematic \\nreview evaluates the current state and future prospects of causal inference \\nmethodologies in enhancing ovarian cancer diagnosis and prognosis. We performed \\na comprehensive systematic review focusing on causal inference methodologies \\napplied to ovarian cancer research. The analysis encompassed biomarker \\nidentification, pathogenic mechanism elucidation, and multimodal data \\nintegration. Additionally, we analyzed the synergistic combination of causal \\ninference with machine learning approaches across genomic, transcriptomic, \\nproteomic, and imaging datasets. Causal inference methods have shown \\neffectiveness in identifying crucial biomarkers and revealing underlying \\npathogenic mechanisms of ovarian cancer. The integration of machine learning \\nwith causal inference has enhanced model interpretability, clinical \\napplicability, and diagnostic-prognostic accuracy. These approaches have \\nachieved improved predictions of disease progression and optimization of \\ntreatment strategies by leveraging clinical, genetic, and imaging data. Causal \\ninference shows considerable potential in advancing precision medicine for \\novarian cancer, offering robust frameworks for addressing confounding factors \\nand establishing causal relationships. As these methodologies evolve and data \\nvolumes expand, their application may become increasingly valuable in oncology \\npractice.\\n\\n© 2025. The Author(s), under exclusive licence to Federación de Sociedades \\nEspañolas de Oncología (FESEO).\\n\\nDOI: 10.1007/s12094-025-03967-1\\nPMID: 40537725\\n\\nConflict of interest statement: Declarations. Conflict of interest: The authors \\ndeclare that they have no known competing financial interests or personal \\nrelationships that could have appeared to influence the work reported in this \\npaper. Ethical approval and consent to participate: Not applicable. Consent for \\npublication: All authors consent to the publication of the manuscript. Clinical \\ntrial number: Not applicable.',\n",
       " \"1. Support Care Cancer. 2025 Jun 19;33(7):599. doi: 10.1007/s00520-025-09659-4.\\n\\nThe effectiveness of an expressive writing diary on preoperative anxiety in \\nbreast cancer patients undergoing surgery: a mixed methods intervention design.\\n\\nWanchitnai N(1), Tongsai S(2), Ratta-Apha W(3), Chimchalong R(1), Chuthapisith \\nS(4), Klankluang W(5).\\n\\nAuthor information:\\n(1)Department of Nursing Siriraj Hospital, Faculty of Medicine Siriraj Hospital, \\nMahidol University, Bangkok, Thailand.\\n(2)Department of Research, Faculty of Medicine Siriraj Hospital, Mahidol \\nUniversity, Bangkok, Thailand.\\n(3)Department of Psychiatry, Faculty of Medicine Siriraj Hospital, Mahidol \\nUniversity, Bangkok, Thailand.\\n(4)Department of Surgery, Faculty of Medicine Siriraj Hospital, Mahidol \\nUniversity, Bangkok, Thailand.\\n(5)Behavioral Science Research Institute, Srinakharinwirot University, 114 Soi \\nSukhumvit 23, Sukhuwit Road, Khlong Toei Nuea, Wattana, Bangkok, 10110, \\nThailand. watanachai.kl@kmitl.ac.th.\\n\\nPURPOSE: This study examined the effectiveness of an expressive writing diary \\n(EWD) in reducing preoperative anxiety among breast cancer patients awaiting \\nsurgery. It also explores patient experiences in both EWD and the relaxing diary \\n(RD) groups, with an emphasis on anxiety levels and writing engagement.\\nMETHODS: We employed a mixed methods intervention design in a randomized \\ncontrolled trial of 30 breast cancer patients. The participants were assigned to \\neither the EWD (intervention) or RD (control) group. Preoperative anxiety levels \\nwere measured via the State-Trait Anxiety Inventory before and after in both \\ngroups. Qualitative data were then analyzed to investigate patient experiences.\\nRESULTS: The EWD and RD significantly reduced preoperative anxiety. However, the \\nEWD yielded a greater reduction (mean difference\\u2009=\\u20095.1; 95% CI\\u2009-\\u20091.7, 11.9). The \\nqualitative findings indicated that the EWD offered a structured outlet for \\nemotional expression. This approach fostered heightened emotional awareness, \\npersonal growth, and transformation. In contrast, RD participants reported \\ndifficulty engaging, largely because of insufficient structure and guidance.\\nCONCLUSION: Both the EWD and RD effectively reduced preoperative anxiety in \\nbreast cancer patients, although the EWD had stronger effects. Moreover, the EWD \\nincreased patients' emotional well-being and quality of life prior to surgery. \\nThus, an EWD may serve as a valuable supplement to preoperative care.\\n\\n© 2025. The Author(s), under exclusive licence to Springer-Verlag GmbH Germany, \\npart of Springer Nature.\\n\\nDOI: 10.1007/s00520-025-09659-4\\nPMID: 40537724\\n\\nConflict of interest statement: Declarations. Ethical approval: The study was \\nconducted in compliance with the Declaration of Helsinki (1996) and adhered to \\nall applicable ethical guidelines. Ethical approval was obtained from the \\nInstitutional Review Board of the Faculty of Medicine Siriraj Hospital, Mahidol \\nUniversity (Reference No. Si-594/2021). Consent to participate: Informed consent \\nwas obtained from all individual participants included in the study. Competing \\ninterests: The authors declare no competing interests.\",\n",
       " '1. Sports Med. 2025 Jun 19. doi: 10.1007/s40279-025-02254-x. Online ahead of\\nprint.\\n\\nHigh-Intensity Interval Training for Cancer Patients: Taking Training \\nProgression, Feasibility, and Patient Preferences Into Account.\\n\\nFranssen RFW(1)(2), van Meeteren NLU(3)(4), Bongers BC(5)(6).\\n\\nAuthor information:\\n(1)Department of Clinical Physical Therapy, VieCuri Medical Center, Tegelseweg \\n210, 5912 BL, Venlo, The Netherlands. rfranssen@viecuri.nl.\\n(2)Department of Epidemiology, GROW Research Institute for Oncology and \\nReproduction, Maastricht University, Maastricht, The Netherlands. \\nrfranssen@viecuri.nl.\\n(3)Department of Anesthesiology, Erasmus Medical Center, Rotterdam, The \\nNetherlands.\\n(4)Top Sector Life Sciences and Health (Health∼Holland), The Hague, The \\nNetherlands.\\n(5)Department of Nutrition and Movement Sciences, Institute of Nutrition and \\nTranslational Research in Metabolism (NUTRIM), Maastricht University, \\nMaastricht, The Netherlands.\\n(6)Department of Surgery, Institute of Nutrition and Translational Research in \\nMetabolism (NUTRIM), Maastricht University, Maastricht, The Netherlands.\\n\\nDOI: 10.1007/s40279-025-02254-x\\nPMID: 40537720\\n\\nConflict of interest statement: Declarations. Funding: No funding was received \\nfor the preparation or publication of this letter. Conflicts of interest: All \\nauthors declare no potential conflicts of interest. Author contributions: \\nManuscript concepts: Ruud F.W. Franssen, Nico L.U. van Meeteren, and Bart C. \\nBongers. Manuscript design: Ruud F.W. Franssen, Nico L.U. van Meeteren, and Bart \\nC. Bongers. Manuscript preparation: Nico L.U. van Meeteren. Manuscript editing: \\nRuud F.W. Franssen, Nico L.U. van Meeteren, and Bart C. Bongers. Manuscript \\nreview: Ruud F.W. Franssen, Nico L.U. van Meeteren, and Bart C. Bongers. All the \\nauthors have made a significant contribution to this manuscript, have seen and \\napproved the final manuscript, and have agreed to its submission to Sports \\nMedicine.',\n",
       " '1. J Gastrointest Cancer. 2025 Jun 19;56(1):139. doi: 10.1007/s12029-025-01249-1.\\n\\nComprehensive Bibliometric Analysis of Prediction Models for HCC: Current Trends \\nand Future Prospects.\\n\\nLi D(1), Sun J(1), Fu X(2), Gao F(3).\\n\\nAuthor information:\\n(1)Department of Biliary and Pancreatic Surgery, Third Hospital of Shanxi \\nMedical University, Shanxi Bethune Hospital, Shanxi Academy of Medical Sciences, \\nTongji Shanxi Hospital, No. 99, Longcheng Street, Xiaodian District, Taiyuan, \\n030032, China.\\n(2)Department of Biliary and Pancreatic Surgery, Third Hospital of Shanxi \\nMedical University, Shanxi Bethune Hospital, Shanxi Academy of Medical Sciences, \\nTongji Shanxi Hospital, No. 99, Longcheng Street, Xiaodian District, Taiyuan, \\n030032, China. fxfyisheng@163.com.\\n(3)Department of Biliary and Pancreatic Surgery, Third Hospital of Shanxi \\nMedical University, Shanxi Bethune Hospital, Shanxi Academy of Medical Sciences, \\nTongji Shanxi Hospital, No. 99, Longcheng Street, Xiaodian District, Taiyuan, \\n030032, China. gfgf2007@163.com.\\n\\nBACKGROUND: Hepatocellular carcinoma (HCC) is the most common primary malignant \\nliver tumor, with rising incidence and mortality rates posing a significant \\nthreat to global public health. Accurate prediction of liver cancer occurrence \\nand progression is essential for improving patient prognosis. This study uses \\nbibliometric methods to analyze the current state and future trends in liver \\ncancer prediction research.\\nMETHODS: A search was conducted in the Web of Science (WOS) database on October \\n22, 2023, identifying 1092 articles on liver cancer prediction. These articles \\nwere quantitatively analyzed using CiteSpace 6.2 software, with a focus on \\nresearch hotspots, authors, countries, and keywords.\\nRESULTS: The study involved 114 countries, 4254 institutions, and 280 journals, \\nwith 48,788 citations. China (826 papers) and the USA (96 papers) dominate the \\nfield. Leading institutions include Sun Yat-sen University, Fudan University, \\nZhejiang University, and Yonsei University. The most cited journals were \\nHepatology (2209 citations) and Journal of Hepatology (946 citations). Frontiers \\nin Oncology had the highest H-index (14). Key authors include Kim Seung Up (23 \\npapers) and Ahn Sang Hoon (H-index\\u2009=\\u200914). Early research focused on risk factors \\nand staging, while recent studies emphasize DNA methylation, immune \\nmicroenvironments, and tumor metastasis. Future research will focus on \\nmulti-omics data integration and AI-driven predictive model optimization.\\nCONCLUSION: This study provides a comprehensive overview of liver cancer \\nprediction research, highlighting key trends and the potential of multi-omics \\ndata and machine learning to enhance predictive models and clinical outcomes.\\n\\n© 2025. The Author(s), under exclusive licence to Springer Science+Business \\nMedia, LLC, part of Springer Nature.\\n\\nDOI: 10.1007/s12029-025-01249-1\\nPMID: 40537718\\n\\nConflict of interest statement: Declarations. Ethics Approval and Consent to \\nParticipate: The present study does not involve humans or animals and does not \\nrequire approval from an ethics committee. We have adhered to relevant ethical \\nguidelines and ensured the honesty and reliability of our research. Consent for \\nPublication: All authors have reviewed and agreed to the publication of this \\nmanuscript. Disclaimer: The authors confirm that the content of this manuscript \\ndoes not infringe upon the rights of any third party. Competing interests: The \\nauthors declare no competing interests.',\n",
       " '1. EJNMMI Res. 2025 Jun 19;15(1):73. doi: 10.1186/s13550-025-01271-1.\\n\\nThe impact of patient hydration and forced diuresis on [(18)F]PSMA-1007 urinary \\nbladder uptake in PET/CT imaging.\\n\\nKhalimon AI(1), Nikiforuk AI(2), Khodzhibekova MM(2), Khamadeeva GF(2), \\nKhodakova DY(2), Lazutina TN(2), Pylova IV(2), Leontyev AV(2), Bobrov VS(2), \\nMukhortova OV(3), Aslanidi IP(3).\\n\\nAuthor information:\\n(1)Department of Nuclear Medicine, P. Hertsen Moscow Oncology Research Institute \\n- The Branch of the FSBI \"National Medical Research Radiological Centre\" of the \\nMinistry of Health of the Russian Federation, Moscow, Russia. \\nmarkyhaws@gmail.com.\\n(2)Department of Nuclear Medicine, P. Hertsen Moscow Oncology Research Institute \\n- The Branch of the FSBI \"National Medical Research Radiological Centre\" of the \\nMinistry of Health of the Russian Federation, Moscow, Russia.\\n(3)Department of Nuclear Medicine, A. N. Bakulev National Medical Research \\nCenter for Cardiovascular Surgery of the Russian Ministry of Health, Moscow, \\nRussia.\\n\\nDOI: 10.1186/s13550-025-01271-1\\nPMID: 40537715\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: The study was conducted at the P. Hertsen Moscow Oncology Research \\nInstitute - the branch of the FSBI “National Medical Research Radiological \\nCentre” of the Ministry of Health of the Russian Federation following the \\nDeclaration of Helsinki and was approved by the local ethical committee. Written \\ninformed consent was obtained from all individual participants included in the \\nstudy. Consent for publication: The authors affirm that human research \\nparticipants provided informed consent for publication of the images in Fig.\\xa03 \\n(a-f). Conflict of interest: The authors declare that they have no conflict of \\ninterest.',\n",
       " '1. Curr Pain Headache Rep. 2025 Jun 19;29(1):94. doi: 10.1007/s11916-025-01409-9.\\n\\nTrialing Strategies Prior to Intrathecal Drug Delivery in Cancer-Related Pain: A \\nNarrative Review.\\n\\nRoux E(1), Kaye AD(2), Shah SS(3), Shekoohi S(4), Hao D(5).\\n\\nAuthor information:\\n(1)Harvard Medical School, Boston, MA, 02215, USA.\\n(2)Departments of Anesthesiology and Pharmacology, Toxicology, and \\nNeurosciences, Louisiana State University Health Sciences Center Shreveport, \\nShreveport, LA, 71103, USA.\\n(3)School of Medicine, Louisiana State University Health Sciences Center at \\nShreveport, Shreveport, LA, 71103, USA.\\n(4)Department of Anesthesiology, Louisiana State University Health Sciences \\nCenter Shreveport, Shreveport, LA, 71103, USA. sahar.shekoohi@lsuhs.edu.\\n(5)Department of Anesthesia, Critical Care, and Pain Medicine, Harvard Medical \\nSchool, Massachusetts General Hospital, Boston, MA, 02114, USA.\\n\\nPURPOSE OF REVIEW: Cancer-related pain poses a significant clinical challenge, \\nespecially in advanced stages where systemic analgesic therapies become \\ninsufficient or intolerable. Intrathecal drug delivery systems (IDDS) offer \\ntargeted pain control while minimizing systemic exposure. However, the optimal \\ntrialing approach before permanent IDDS implantation remains contentious. This \\nnarrative review examines literature on IDDS trialing strategies in cancer pain \\nmanagement. A comprehensive search was conducted of PubMed, MEDLINE, and Embase \\ndatabases and identified studies published up to January 2025. The review \\nincluded prospective and retrospective studies, randomized controlled trials, \\ncohort studies, and case series on trialing techniques, clinical outcomes, \\nsafety, tolerability, and efficacy. Key strateghies assessed include single-shot \\nintrathecal bolus, multiple intrathecal boluses, continuous epidural infusion, \\nand continuous intrathecal infusion. The review found significant variability in \\ntrialing practices, with limited high-quality comparative data to support \\nstandardized protocols. Trial success criteria varied widely, encompassing pain \\nreduction, side effects, and patient-reported outcomes.\\nRECENT FINDINGS: The studies described a range of trialing strategies with \\nvarying durations, opioid dosages, and criteria for success. However, due to the \\nlack of direct comparisons between these approaches, it is difficult to draw \\nclear conclusions about the relative effectiveness of continuous intrathecal, \\ncontinuous epidural, and bolus-based trials. Some institutions bypassed \\ntrialing, prioritizing symptom relief over procedural risks.\\nCONCLUSION: This review highlights the need for individualized trialing \\nstrategies based on patient status, institutional preferences, and clinician \\nexpertise. Given the variability in current practices, further research is \\nneeded to establish evidence-based guidelines and optimize clinical \\ndecision-making.\\n\\n© 2025. The Author(s), under exclusive licence to Springer Science+Business \\nMedia, LLC, part of Springer Nature.\\n\\nDOI: 10.1007/s11916-025-01409-9\\nPMID: 40537714\\n\\nConflict of interest statement: Declarations. Compliance with Ethical \\nGuidelines: This article is based on previous studies and contains no new \\nstudies with human participants or animals performed by any authors. Competing \\nInterests: ADK is the editor in Chief for CPHRs.',\n",
       " '1. J Mol Neurosci. 2025 Jun 20;75(3):79. doi: 10.1007/s12031-025-02361-4.\\n\\nCerebrospinal Fluid-Derived Genomic Alterations Tracking Glioma.\\n\\nZhu Z(#)(1), Yu Y(#)(2), Liu F(#)(3), Yang X(#)(4), Zhou Z(5), Zhu Z(6), Wang \\nZ(7), Zhang J(8).\\n\\nAuthor information:\\n(1)Department of Intensive Care Unit, State Key Laboratory of Oncology in South \\nChina, Guangdong Provincial Clinical Research Center for Cancer, Sun Yat-Sen \\nUniversity Cancer Center, Guangzhou, China.\\n(2)Department of Neurosurgery, State Key Laboratory of Oncology in South China, \\nGuangdong Provincial Clinical Research Center for Cancer, Sun Yat-Sen University \\nCancer Center, Guangzhou, China.\\n(3)Department of Neurosurgery, Tumor Hospital Affiliated of Xinjiang Medical \\nUniversity, Xinjiang, China.\\n(4)Department of Anesthesiology, State Key Laboratory of Oncology in South \\nChina, Guangdong Provincial Clinical Research Center for Cancer, Sun Yat-Sen \\nUniversity Cancer Center, Guangzhou, China.\\n(5)Genetron Health (Beijing) Co. Ltd, Beijing, China.\\n(6)Department of Neurosurgery, Tumor Hospital Affiliated of Xinjiang Medical \\nUniversity, Xinjiang, China. 404731230@qq.com.\\n(7)Department of Experimental Research, Sun Yat-Sen University Cancer Center, \\nState Key Laboratory of Oncology in South China, Guangdong Provincial Clinical \\nResearch Center for Cancer, Sun Yat-Sen University Cancer Center, Guangzhou, \\nChina. wangzf@sysucc.org.cn.\\n(8)Department of Neurosurgery, State Key Laboratory of Oncology in South China, \\nGuangdong Provincial Clinical Research Center for Cancer, Sun Yat-Sen University \\nCancer Center, Guangzhou, China. zhangji@sysucc.org.cn.\\n(#)Contributed equally\\n\\nDiffuse gliomas are aggressive brain tumors known for heterogeneity and frequent \\noncogenic mutations. Our study harnessed circulating tumor DNA (ctDNA) in \\ncerebrospinal fluid (CSF) as a less invasive method for disease monitoring and \\nguiding therapeutic interventions. Through targeted sequencing of ctDNA from CSF \\nand matched blood and tumor tissue samples, we aimed to identify \\nglioma-associated somatic alterations and DNA fragmentations. The identified \\nglioma-associated mutations from ctDNA in CSF and genome DNA of the resected \\ntumor were compared, revealed a broad genetic alteration spectrum within CSF \\nctDNA, closely reflecting the genomic profiles of corresponding tumor samples. \\nAnd we found 91.67% (11/12) of tissue samples analyzed by next-generation \\nsequencing (NGS), a minimum of one tumor-specific mutation was present. Also, at \\nleast one tumor-specific mutation was detected in 91.67% of serial CSF ctDNA \\nsamples (11/12). In some patients, CSF sequencing showed higher mutation \\ndetection rate compared to tissue sequencing. Moreover, the average mutation \\nfrequencies were similar between CSF and tumor tissue samples. These results \\nsupport that CSF ctDNA is a reliable candidate for detecting glioma-specific \\nalterations for molecular profiling, complements the histopathological, \\nmolecular and imaging defect, and can be adopted into clinical practice.\\n\\n© 2025. The Author(s), under exclusive licence to Springer Science+Business \\nMedia, LLC, part of Springer Nature.\\n\\nDOI: 10.1007/s12031-025-02361-4\\nPMID: 40537713\\n\\nConflict of interest statement: Declarations. Ethics Approval: Approval of the \\nresearch protocol by an institutional review board: All procedures performed in \\nthe study were approved by the Ethics Committee and Institutional Review Board \\nof Affiliated Tumor Hospital of Xinjiang Medical University and Sun Yat-sen \\nUniversity Cancer Center (No. GZR2018-244). The project was conducted in \\naccordance with Helsinki Declaration ethical standards. Informed Consent: \\nWritten informed consents were obtained from their family numbers or \\nparticipants. Consent for Publication: All the authors hereby collectively grant \\ntheir consent for the publication of this manuscript. Competing Interests: The \\nauthors declare no competing interests.',\n",
       " \"1. Mol Imaging Biol. 2025 Jun 19. doi: 10.1007/s11307-025-02025-0. Online ahead\\nof  print.\\n\\nPoor Diagnostic Performance of the Melanin-Binding Tracer [18 F]MEL050 in Human \\nMelanoma Indicates Biological Heterogeneity.\\n\\nWare RE(#)(1), Kee D(#)(2), Roselt P(3), Greguric I(4), Katsifis A(5)(6), \\nBourdier T(5)(6), Noonan W(3), Murray W(7), Mitchell C(7), Downes M(8), \\nShackleton M(9)(10), McArthur GA(2)(11), Hicks RJ(12)(13)(14).\\n\\nAuthor information:\\n(1)The Melbourne Theranostic Innovation Centre, Level 8, 14-20 Blackwood St, \\nNorth, Melbourne, VIC, 3051, Australia.\\n(2)The Skin and Melanoma Service, the Peter MacCallum Cancer Centre, 305 Grattan \\nSt, Melbourne, VIC, 3000, Australia.\\n(3)Department of Radiopharmaceutical Sciences, Cancer Imaging, the Peter \\nMacCallum Cancer Centre, 305 Grattan St, Melbourne, VIC, 3000, Australia.\\n(4)Australian Nuclear Science and Technology Organisation, Lucas Heights, NSW, \\nAustralia.\\n(5)Department of Molecular Imaging, Royal Prince Alfred Hospital, Camperdown, \\nSydney, NSW, Australia.\\n(6)Pharmacy School, University of Sydney, Sydney, NSW, Australia.\\n(7)The Department of Pathology, the Peter MacCallum Cancer Centre, 305 Grattan \\nSt, Melbourne, VIC, 3000, Australia.\\n(8)The University of Melbourne, Clinical Epidemiology & Biostatistics Unit, \\nMurdoch Children's Research Institute, Parkville, Australia.\\n(9)School of Translational Medicine, Monash University, Prahran, VIC, 3004, \\nAustralia.\\n(10)Department of Oncology, Alfred Health, Prahran, VIC, 3004, Australia.\\n(11)Sir Peter MacCallum Department of Oncology, the University of Melbourne, \\nParkville, Australia.\\n(12)The Melbourne Theranostic Innovation Centre, Level 8, 14-20 Blackwood St, \\nNorth, Melbourne, VIC, 3051, Australia. rod.hicks@premit.net.au.\\n(13)School of Translational Medicine, Monash University, Prahran, VIC, 3004, \\nAustralia. rod.hicks@premit.net.au.\\n(14)The Department of Medicine, St Vincent's Medical School, the University of \\nMelbourne, Fitzroy, Australia. rod.hicks@premit.net.au.\\n(#)Contributed equally\\n\\nPURPOSE: Malignant melanoma is a highly lethal malignancy typically \\ncharacterized by the expression of melanin, which is an attractive diagnostic \\nand therapeutic target in these cancers because it is expressed in few other \\ntissues. Following preclinical evaluation of the melanin-targeting PET tracer, \\n[18F]-6-fluoro-N-[2-(diethylamino)ethyl] pyridine-3-carboxamide ([18F]MEL050), \\nwe sought to evaluate this agent in patients with melanoma.\\nMETHOD: A phase I clinical trial was performed in ten patients with metastatic \\nmelanoma. Safety, dosimetry and diagnostic performance of intravenously \\nadministered][18F]MEL050 were evaluated. Based on results from this trial, we \\nfurther assessed the prevalence and prognostic significance of loss of melanin \\nexpression in two historical patient cohorts for which there were matching \\nhistological and clinical outcome data.\\nRESULTS: Across the trial cohort, no adverse safety signals resulted from \\n[18F]MEL050 administration. The whole-body effective dose was 0.0163 mSV/MBq for \\nan adult male and 0.0206 mSV/MBq for an adult female. The human biodistribution \\nwas favorable with low uptake in organs at high risk of metastatic spread, \\nincluding the brain. Of metastatic sites identified as melanoma on [18F]FDG \\nPET/CT, only 31/65 (48%) were positive on [18F]MEL050 PET. Four \\n[18F]FDG+[18F]MEL050+ metastases were resected from three patients and found to \\nbe melanotic by histological examination, whereas five [18F]FDG+[18F]MEL050- \\nmetastases from two patients were amelanotic. In our historical cohorts, \\namelanosis was more common in metastatic than primary disease (45% versus 20%) \\nand the presence of melanin within sentinel lymph node metastases was associated \\nwith worse disease-free (HR 2.3 95% CI 1.3 - 4.3, p = 0.002) and \\ndisease-specific survivals (HR 3.6, 95% CI 1.4 - 9.7,p = 0.009) in stage III \\ndisease, compared with amelanotic sentinel lymph node metastases.\\nCONCLUSION: We propose caution in the use of melanin-targeted agents for \\nmelanoma diagnosis and therapy until their utility as prognostic or predictive \\nimaging biomarkers, and the biological implications of loss of melanin \\ndeposition during melanoma progression, are better understood.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s11307-025-02025-0\\nPMID: 40537707\\n\\nConflict of interest statement: Declarations. Conflicts of interest: No authors \\ndeclare conflicts of interest relevant to this manuscript. Ethics approvals: \\nPMCC HREC # 10/11 PMCC HREC #12/74\",\n",
       " '1. Environ Sci Pollut Res Int. 2025 Jun 19. doi: 10.1007/s11356-025-36593-z.\\nOnline  ahead of print.\\n\\nCarcinogenic effects of paraquat-based herbicides: a comprehensive review.\\n\\nGarcía Carrillo M(1), Madanes D(2), Blaustein M(3)(4).\\n\\nAuthor information:\\n(1)Instituto de Biociencias, Biotecnología y Biología Traslacional (iB3), \\nUniversidad de Buenos Aires (UBA), Facultad de Ciencias Exactas y Naturales, \\nDepartamento de Fisiología, Biología Molecular y Celular (DFBMC), Buenos Aires, \\nArgentina. mercedesgcarrillo@fbmc.fcen.uba.ar.\\n(2)Instituto Nacional de Parasitología \"Dr. Mario Fatala Chaben\", Administración \\nNacional de Laboratorios e Institutos de Salud (ANLIS) Malbrán, Buenos Aires, \\nArgentina.\\n(3)Instituto de Biociencias, Biotecnología y Biología Traslacional (iB3), \\nUniversidad de Buenos Aires (UBA), Facultad de Ciencias Exactas y Naturales, \\nDepartamento de Fisiología, Biología Molecular y Celular (DFBMC), Buenos Aires, \\nArgentina.\\n(4)Consejo Nacional de Investigaciones Científicas y Técnicas (CONICET), Buenos \\nAires, Argentina.\\n\\nHerbicides are agrochemicals used for the eradication of weeds in agriculture. \\nFarmers worldwide recognize paraquat-based herbicides for their high \\neffectiveness against different weeds. Previous research indicates that the \\nextensive application of herbicides may endanger environmental, animal, and \\nhuman health due to their toxicity. Specifically, herbicides have been \\nassociated with carcinogenic effects. Several molecular and cellular mechanisms \\nhave been linked to the carcinogenicity of herbicides, including the induction \\nof DNA damage and/or oxidative stress through the generation of reactive oxygen \\nspecies (ROS). Researchers have linked several molecular and cellular mechanisms \\nto the carcinogenicity of herbicides, which would result in the proliferation, \\nsurvival, and invasion of cancerous cells. Particularly, paraquat-based \\nherbicides can disrupt the cellular redox balance, which makes them extremely \\ntoxic to animals and humans. Although the World Health Organization\\'s (WHO) \\nInternational Agency for Research on Cancer (IARC) has not evaluated the \\ncarcinogenicity of paraquat to date, several countries have banned its use due \\nto its association with a high mortality rate from self-poisoning incidents. Our \\nreview seeks to provide a comprehensive panorama of the carcinogenic effects of \\nparaquat-based herbicides in animals and humans, encompassing studies on \\nenvironmental persistence, molecular and cellular mechanisms of toxicity, and \\nepidemiological research on paraquat\\'s association with carcinogenesis.\\n\\n© 2025. The Author(s), under exclusive licence to Springer-Verlag GmbH Germany, \\npart of Springer Nature.\\n\\nDOI: 10.1007/s11356-025-36593-z\\nPMID: 40537706\\n\\nConflict of interest statement: Declarations. Ethical approval: This is not \\napplicable. Consent to participate: This is not applicable. Consent for \\npublication: This is not applicable. Competing interests: The authors declare no \\ncompeting interests. Declaration of generative AI in scientific writing: The \\nauthors declare that generative AI and AI-assisted technologies were used in the \\nwriting process to improve the readability and language of the manuscript under \\ncareful human oversight.',\n",
       " '1. Geroscience. 2025 Jun 19. doi: 10.1007/s11357-025-01736-0. Online ahead of \\nprint.\\n\\nNRF2 pathway activation predicts poor prognosis in lung cancer: a cautionary \\nnote on antioxidant interventions.\\n\\nUngvari Z(#)(1)(2)(3)(4)(5), Menyhart O(#)(6)(7)(8), Lehoczki A(#)(9)(10)(11), \\nFekete M(12)(13), Fazekas-Pongor V(12)(13), Ocana A(14)(15), Varga P(12)(13), \\nGyőrffy B(6)(7)(8)(16).\\n\\nAuthor information:\\n(1)Vascular Cognitive Impairment, Neurodegeneration and Healthy Brain Aging \\nProgram, Department of Neurosurgery, University of Oklahoma Health Sciences \\nCenter, Oklahoma City, OK, USA.\\n(2)Stephenson Cancer Center, University of Oklahoma, Oklahoma City, OK, USA.\\n(3)Oklahoma Center for Geroscience and Healthy Brain Aging, University of \\nOklahoma Health Sciences Center, Oklahoma City, OK, USA.\\n(4)Department of Health Promotion Sciences, College of Public Health, University \\nof Oklahoma Health Sciences Center, Oklahoma City, OK, USA.\\n(5)International Training Program in Geroscience, Doctoral College/Institute of \\nPreventive Medicine and Public Health, Semmelweis University, Budapest, Hungary.\\n(6)Cancer Biomarker Research Group, Institute of Molecular Life Sciences, \\nHungarian Research Network, Magyar Tudósok Körútja 2, Budapest, 1117, Hungary.\\n(7)National Laboratory for Drug Research and Development, Magyar Tudósok Körútja \\n2, Budapest, 1117, Hungary.\\n(8)Department of Bioinformatics, Semmelweis University, Budapest, H-1094, \\nHungary.\\n(9)Doctoral College, Health Sciences Division, Semmelweis University, Budapest, \\nHungary. Andrea.M.Lehoczki@gmail.com.\\n(10)Institute of Preventive Medicine and Public Health, Semmelweis University, \\nBudapest, Hungary. Andrea.M.Lehoczki@gmail.com.\\n(11)Fodor Center for Prevention and Healthy Aging, Semmelweis University, \\nBudapest, Hungary. Andrea.M.Lehoczki@gmail.com.\\n(12)Institute of Preventive Medicine and Public Health, Semmelweis University, \\nBudapest, Hungary.\\n(13)Fodor Center for Prevention and Healthy Aging, Semmelweis University, \\nBudapest, Hungary.\\n(14)Experimental Therapeutics in Cancer Unit, Instituto de Investigación \\nSanitaria San Carlos (IdISSC), and CIBERONC, Madrid, Spain.\\n(15)INTHEOS-CEU-START Laboratory, Facultad de Medicina, Universidad CEU San \\nPablo, 28668 Boadilla del Monte, Madrid, Spain.\\n(16)Department of Biophysics, Medical School, University of Pecs, Pecs, H-7624, \\nHungary.\\n(#)Contributed equally\\n\\nLung cancer is a leading cause of cancer-related mortality worldwide. As an \\nage-related disease, its pathogenesis is shaped by several molecular hallmarks \\nof aging, including impaired DNA repair and diminished stress resilience. The \\ntranscription factor NRF2 (nuclear factor erythroid 2-related factor 2) is a \\nmaster regulator of oxidative stress defense and cellular survival. While NRF2 \\nactivation is protective in aging tissues, it may also be exploited by cancer \\ncells to promote tumor progression and therapy resistance. This study aims to \\nevaluate whether NRF2 pathway activation predicts clinical outcomes in lung \\ncancer, with potential implications for the use of NRF2-inducing compounds. We \\nanalyzed transcriptomic and survival data from 2167 lung cancer patients using \\nthe KM Plotter database. A validated 14-gene NRF2 activation signature was used \\nto stratify tumors by NRF2 pathway activity. Associations with overall survival \\n(OS), first progression (FP), and post-progression survival (PPS) were assessed \\nusing Cox regression models and Kaplan-Meier analysis. High NRF2 signature \\nexpression was significantly associated with poorer OS (HR = 1.59, p = 1.3E-9), \\nFP (HR = 1.61, p = 2.6E-5), and PPS (HR = 1.6, p = 0.002). The negative \\nprognostic effect was most pronounced in patients with adenocarcinoma, \\nnode-negative disease, and in female patients. These findings highlight the dual \\nrole of NRF2 in promoting stress resilience and enabling cancer cell survival. \\nNRF2 activation is a predictor of poor clinical outcomes in lung cancer. Given \\nthe widespread use of NRF2-inducing compounds such as resveratrol and \\nsulforaphane, these findings raise important concerns about their safety in \\nindividuals at risk for or living with cancer. Our results underscore the \\nimportance of context-specific evaluation of NRF2-targeted interventions and \\ncaution against the indiscriminate use of NRF2-activating agents in aging \\npopulations, particularly in individuals at risk for lung cancer.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s11357-025-01736-0\\nPMID: 40537696\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: NA Consent for publication: NA Competing interests: Dr. Balazs \\nGyorffy serves as Associate Editor for GeroScience. Dr. Zoltan Ungvari serves as \\nEditor-in-Chief for GeroScience.',\n",
       " \"1. Discov Oncol. 2025 Jun 19;16(1):1159. doi: 10.1007/s12672-025-02961-x.\\n\\nALOX15B-based efferocytosis clusters: prognostic implications and immune cell \\ninfiltration in breast cancer.\\n\\nChen J(1), Zheng C(2), Zhang F(3), Sun S(2), Liu G(4), Huang G(5).\\n\\nAuthor information:\\n(1)Department of Oncology, Shenzhen Key Laboratory of Gastrointestinal Cancer \\nTranslational Research, Cancer Institute, Xinhua Hospital, Peking University \\nShenzhen Hospital, Shenzhen, 518036, China.\\n(2)Department of Thyroid and Breast Surgery, Shenzhen Second People's Hospital, \\nThe First Affiliated Hospital of Shenzhen University, Shenzhen, 518035, \\nGuangdong, China.\\n(3)Department of Pathology, Youxian People's Hospital, Zhuzhou, 412300, Hunan, \\nChina.\\n(4)Department of Thyroid and Breast Surgery, Shenzhen Second People's Hospital, \\nThe First Affiliated Hospital of Shenzhen University, Shenzhen, 518035, \\nGuangdong, China. liugw8318@163.com.\\n(5)Department of Thyroid and Breast Surgery, Shenzhen Second People's Hospital, \\nThe First Affiliated Hospital of Shenzhen University, Shenzhen, 518035, \\nGuangdong, China. huangguo@usc.edu.cn.\\n\\nOBJECTIVE: Efferocytosis, a process of apoptotic cell clearance, is implicated \\nin the initiation and progression of breast cancer. This study aims to construct \\nan efferocytosis-related genetic risk model to evaluate its prognostic value and \\nits association with immune infiltration in breast cancer.\\nMETHODS: The expression patterns of efferocytosis-related genes in breast cancer \\nwere analyzed, followed by clustering analysis to explore their relationship \\nwith tumor immune infiltration across different subtypes. A risk model based on \\nefferocytosis-related genes was developed to predict breast cancer prognosis and \\nassess its correlation with immune infiltration. Using this model, the \\nassociation between ALOX15B expression and breast cancer survival was evaluated. \\nIn Vitro Functional Assays, the biological role of ALOX15B in breast cancer \\nprogression was investigated using the following approaches: Cellular \\nProliferation: MTT assay, EdU incorporation assay, and colony formation assay \\nwere performed to assess proliferation in MCF7 and MDA-MB-231 cells. Transwell \\nwas used to quantify migratory and invasive. Flow cytometry was conducted to \\nevaluate cell cycle distribution. Western blotting was performed to assess \\nEpithelial-Mesenchymal Transition marker expression.\\nRESULTS: Clustering analysis of the expression of efferocytosis-related genes in \\nbreast cancer, Cluster A had a good prognosis and was closely associated with \\nimmune cell infiltration. Functional enrichment analysis of the activation and \\nadaptation of immune response in Cluster A clustering may be an important factor \\nin improving breast cancer prognosis. In addition, an immune-related gene risk \\nmodel was constructed, which was found to be a valid predictor of breast cancer \\nprognosis. Finally, it was highlighted that the expression of ALOX15B inhibited \\nthe proliferation, movement, invasion, and DNA replication capability of breast \\ncancer cells. The growth of breast cancer cells was also impeded by ALOX15B, \\nwhich hindered cell division during the G1 phase. The expression of ALOX15B is \\nassociated with the sensitivity of a variety of chemotherapeutic agents.\\nCONCLUSION: Efferocytosis-related models can be used for the assessment of \\nbreast cancer prognosis, while the efferocytosis-related gene ALOX15B can affect \\nthe migration and invasive ability of breast cells.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s12672-025-02961-x\\nPMID: 40537687\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: All experimental procedures were approved by the Ethics Committee \\nof The First Affiliated Hospital of Shenzhen University. Consent for \\npublication: All author knows the situation and agrees to publish. Competing \\ninterests: The authors declare no competing interests.\",\n",
       " '1. Med Oncol. 2025 Jun 19;42(7):274. doi: 10.1007/s12032-025-02834-x.\\n\\nRevolutionizing early HCC detection: groundbreaking validation of the HCC-check \\nindex for small tumors in Egyptian patients using CLIP and BCLC staging systems.\\n\\nAttallah KA(1)(2), Farid K(3), Albannan MS(4).\\n\\nAuthor information:\\n(1)Clinical Research Department, Damietta Directorate for Health Affairs, \\nEgyptian Ministry of Health and Population, Damietta, Egypt. \\nk.a.m.attallah@gmail.com.\\n(2)Research and Development Department, Biotechnology Research Center, 23 July \\nSt., Industrial Zone, New Damietta, 34517, Egypt. k.a.m.attallah@gmail.com.\\n(3)Faculty of Medicine, Mansoura University, Mansoura, Egypt.\\n(4)Research and Development Department, Biotechnology Research Center, 23 July \\nSt., Industrial Zone, New Damietta, 34517, Egypt.\\n\\nHepatocellular carcinoma (HCC) is often associated with better treatment options \\nfor small asymptomatic tumors if HCC is caught early. A year ago, the \\n\"Technology in Cancer Research and Treatment\" Journal published a report on the \\nhighly sensitive HCC-Check for the early diagnosis of HCC. Here, we examine and \\nconfirm the applicability of HCC-Check in 486 liver-diseased patients with \\nearlier stages of HCC based on two widely used Cancer of the Liver Italian \\nProgram (CLIP) and Barcelona Clinic Liver Cancer (BCLC) staging systems for the \\nidentification of small tumors. Our results showed that HCC-Check produced 0.90 \\nAUC with 78.26% sensitivity and 83.89% specificity, when used to distinguish \\nsmall HCCs from cirrhotic patients. Our findings showed that HCC-Check increased \\nwith disease progression according to the CLIP and BCLC systems, producing \\nsignificant (p\\u2009<\\u20090.0001) Spearman correlation coefficients of 0.633 and 0.458, \\nrespectively. HCC-Check yielded 0.85 (0.78-0.92) AUC, 66.67% (47.19-82.71) \\nsensitivity, and 83.89% (77.69-88.94) specificity for identifying CLIP (0-1) \\nwhile producing 0.87 (0.79-0.94) AUC, 65% (40.78-84.61) sensitivity, and 83.89% \\n(77.69-88.94) specificity for identifying BCLC A. Additionally, HCC-Check \\nyielded 0.90 (0.83-0.97) AUC, 78.26% (56.30-92.54) sensitivity, and 83.89% \\n(77.69-88.94) specificity for discriminating small tumors\\u2009<\\u20093 cm. Simplified \\nHCC-Check still competes with HCC-Check, providing 0.89 (0.83-0.95) AUC with \\n73.91% (51.60-89.77) sensitivity and 82.51% (76.22-87.72) specificity for \\nidentifying small-sized HCC. All these results were superior to those obtained \\nby AFP. In conclusion, the study findings support that HCC-Check and its \\nsimplified version could be a useful tool for early detection of HCC, aiding \\ndefinitive treatment.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s12032-025-02834-x\\nPMID: 40537682\\n\\nConflict of interest statement: Declarations. Conflict of interest: The authors \\ndeclare that they have no conflict of interest. Consent to publish: All authors \\napproved the manuscript and gave their consent for submission and publication.',\n",
       " '1. Ann Surg Oncol. 2025 Jun 19. doi: 10.1245/s10434-025-17676-0. Online ahead of \\nprint.\\n\\nASO Author Reflections: From BRPC to LAPC-Rethinking Resectability in Pancreatic \\nCancer.\\n\\nGhotbi J(1), Farnes I(2), Labori KJ(2).\\n\\nAuthor information:\\n(1)Department of Hepato-Pancreato-Biliary (HPB) Surgery, Oslo University \\nHospital, Institute of Clinical Medicine, University of Oslo, Oslo, Norway. \\njacob.ghotbi@ous-hf.no.\\n(2)Department of Hepato-Pancreato-Biliary (HPB) Surgery, Oslo University \\nHospital, Institute of Clinical Medicine, University of Oslo, Oslo, Norway.\\n\\nDOI: 10.1245/s10434-025-17676-0\\nPMID: 40537678\\n\\nConflict of interest statement: Disclosure: The authors declare no conflicts of \\ninterest.',\n",
       " '1. Ann Surg Oncol. 2025 Jun 19. doi: 10.1245/s10434-025-17673-3. Online ahead of \\nprint.\\n\\nPatterns of Site and Timing of Recurrence After Curative Resection in Single and \\nMultiple Large Hepatocellular Carcinoma: A Multicenter International \\nComprehensive Analysis.\\n\\nGiannone F(1)(2)(3), Cassese G(4)(5)(6)(7), Cubisino A(8), Felli E(9), Cipriani \\nF(10), Branciforte B(11), Rhaiem R(12), Tropea A(13), Muttillo EM(13), Scarinci \\nA(14), Al Taweel B(15), Brustia R(16)(17), Salame E(9), Sommacale D(16)(17), \\nGruttadauria S(13)(18), Piardi T(12), Grazi GL(19), Torzilli G(11), Aldrighetti \\nL(10), Lesurtel M(8), Han HS(7), Panaro F(4)(5)(6)(15), Pessaux P(20)(21).\\n\\nAuthor information:\\n(1)Hepato-Pancreato-Biliary, Oncologic and Robotic Unit, Azienda \\nOspedaliero-Universitaria SS. Antonio e Biagio e Cesare Arrigo, Alessandria, \\nItaly. giannone.cf@gmail.com.\\n(2)Robotic and HPB Research Unit, Research and Innovation Department (DAIRI), \\nAzienda Ospedaliero-Universitaria SS. Antonio e Biagio e Cesare Arrigo, \\nAlessandria, Italy. giannone.cf@gmail.com.\\n(3)Department of Visceral and Digestive Surgery, University Hospital of \\nStrasbourg, Strasbourg, France. giannone.cf@gmail.com.\\n(4)Hepato-Pancreato-Biliary, Oncologic and Robotic Unit, Azienda \\nOspedaliero-Universitaria SS. Antonio e Biagio e Cesare Arrigo, Alessandria, \\nItaly.\\n(5)Robotic and HPB Research Unit, Research and Innovation Department (DAIRI), \\nAzienda Ospedaliero-Universitaria SS. Antonio e Biagio e Cesare Arrigo, \\nAlessandria, Italy.\\n(6)Department of Health Sciences, University of Eastern Piedmont \"Amedeo \\nAvogadro\", Alessandria, Italy.\\n(7)Department of Surgery, Division of Hepato-Pancreato-Biliary Surgery, Seoul \\nNational University Bundang Hospital, Seongnam, South Korea.\\n(8)Department of HPB Surgery and Liver Transplantation, Beaujon Hospital, APHP, \\nUniversity of Paris Cité, Clichy, France.\\n(9)Liver Transplant and Surgery Department, Trousseau Hospital, Tours, France.\\n(10)Hepatobiliary Surgery Division, IRCCS San Raffaele Scientific Institute, \\nMilan, Italy.\\n(11)Division of Hepatobiliary and General Surgery, Department of Surgery, \\nHumanitas University, Humanitas Clinical and Research Center - IRCCS, Rozzano, \\nMilan, Italy.\\n(12)Department of Oncological Digestive Surgery, Hepatobiliary and Pancreatic \\nSurgery Unit, University Reims Champagne-Ardenne, Reims, France.\\n(13)Department for the Treatment and Study of Abdominal Diseases and Abdominal \\nTransplantation, IRCCS-ISMETT, UPMC (University of Poittsburgh Medical Center), \\nPalermo, Italy.\\n(14)Surgical and Medical Department of Translational Medicine, Sant\\'Andrea \\nHospital, Sapienza University of Rome, Rome, Italy.\\n(15)Department of Surgery, Division of HBP Surgery and Transplantation, \\nSaint-Eloi Hospital, University Hospital of Montpellier, Montpellier, France.\\n(16)Department of Digestive and Hepato-pancreatic-biliary Surgery, AP-HP, \\nHôpital Henri-Mondor, Paris Est Créteil University, UPEC, Créteil, France.\\n(17)Team \"Pathophysiology and Therapy of Chronic Viral Hepatitis and Related \\nCancers\", INSERM U955, Créteil, France.\\n(18)Department of Surgery and Medical and Surgical Specialties, University of \\nCatania, Catania, Italy.\\n(19)Chirurgia Epatobiliopancreatica, AOU Careggi, Florence, Italy.\\n(20)Department of Visceral and Digestive Surgery, University Hospital of \\nStrasbourg, Strasbourg, France.\\n(21)Université de Strasbourg, Inserm, Institut de Recherche sur les Maladies \\nVirales et Hépatiques, U1110, Strasbourg, France.\\n\\nBACKGROUND: Curative options for large hepatocellular carcinoma (LHCC) are \\nlimited because of the high risk of early and extrahepatic recurrence. However, \\nonly a few studies report data on long-term outcomes in large cohorts of \\nresected LHCC. We therefore investigated timing and site of recurrence of LHCC \\nand assessed factors strictly associated with adverse patterns.\\nPATIENTS AND METHODS: This was a retrospective, international, multicenter study \\nof patients undergoing anatomic resection of HCC ≥\\u20095\\xa0cm at 12 hepatobiliary \\nhigh-volume centers. Extrahepatic recurrence was defined as any distant site of \\nmetastasis, while recurrence within 2\\xa0years after surgery was classified as \\nearly recurrence.\\nRESULTS: A total of 869 patients were included. Recurrence was observed in 487 \\n(56%) cases. Patterns associated with reduced overall survival were early \\n(p\\u2009<\\u20090.001) and simultaneous intrahepatic and extrahepatic recurrence \\n(p\\u2009=\\u20090.038). Variables independently associated with early recurrence were age \\n(p\\u2009=\\u20090.037), major hepatectomy (p\\u2009=\\u20090.023), microvascular invasion (p\\u2009=\\u20090.011), \\nsatellites nodules (p\\u2009=\\u20090.005), and open approach (p\\u2009=\\u20090.025). Variables \\ncorrelated with simultaneous intra- and extrahepatic relapse were age \\n(p\\u2009<\\u20090.001), preoperative transarterial chemoembolization (TACE) (p\\u2009<\\u20090.001), \\nmicrovascular invasion (p\\u2009<\\u20090.001), and satellite nodules (p\\u2009=\\u20090.026).\\nCONCLUSIONS: Surgery for LHCC is associated with a high risk of early \\nrecurrence. Apart from pathological variables, factors independently associated \\nwith adverse patterns were open approach and age for the early recurrence and \\npreoperative TACE and age for simultaneous intra-extrahepatic relapse. BCLC \\nstage was not associated with timing nor with site of recurrence.\\n\\n© 2025. Society of Surgical Oncology.\\n\\nDOI: 10.1245/s10434-025-17673-3\\nPMID: 40537677\\n\\nConflict of interest statement: Disclosure: The authors have no related \\nconflicts of interest or fundings to declare.',\n",
       " \"1. EMBO Mol Med. 2025 Jun 20. doi: 10.1038/s44321-025-00258-8. Online ahead of \\nprint.\\n\\nReplicated blood-based biomarkers for myalgic encephalomyelitis not explicable \\nby inactivity.\\n\\nBeentjes SV(#)(1)(2)(3), Miralles Méharon A(4), Kaczmarczyk J(5), Cassar A(4), \\nSamms GL(6), Hejazi NS(7), Khamseh A(#)(8)(9)(10), Ponting CP(#)(11).\\n\\nAuthor information:\\n(1)School of Mathematics and Maxwell Institute for Mathematical Sciences, \\nUniversity of Edinburgh, Edinburgh, EH9 3FD, UK. Sjoerd.Beentjes@ed.ac.uk.\\n(2)MRC Human Genetics Unit, Institute of Genetics & Cancer, University of \\nEdinburgh, Edinburgh, EH4 2XU, UK. Sjoerd.Beentjes@ed.ac.uk.\\n(3)Division of Biostatistics, University of California, Berkeley, CA, \\n94720-7360, USA. Sjoerd.Beentjes@ed.ac.uk.\\n(4)School of Informatics, University of Edinburgh, Edinburgh, EH8 9AB, UK.\\n(5)School of Mathematics and Maxwell Institute for Mathematical Sciences, \\nUniversity of Edinburgh, Edinburgh, EH9 3FD, UK.\\n(6)MRC Human Genetics Unit, Institute of Genetics & Cancer, University of \\nEdinburgh, Edinburgh, EH4 2XU, UK.\\n(7)Harvard T.H. Chan School of Public Health, Department of Biostatistics, \\nBoston, MA, 02115, USA.\\n(8)MRC Human Genetics Unit, Institute of Genetics & Cancer, University of \\nEdinburgh, Edinburgh, EH4 2XU, UK. ava.khamseh@ed.ac.uk.\\n(9)Division of Biostatistics, University of California, Berkeley, CA, \\n94720-7360, USA. ava.khamseh@ed.ac.uk.\\n(10)School of Informatics, University of Edinburgh, Edinburgh, EH8 9AB, UK. \\nava.khamseh@ed.ac.uk.\\n(11)MRC Human Genetics Unit, Institute of Genetics & Cancer, University of \\nEdinburgh, Edinburgh, EH4 2XU, UK. Chris.Ponting@ed.ac.uk.\\n(#)Contributed equally\\n\\nMyalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) is a common \\nfemale-biased disease. ME/CFS diagnosis is hindered by the absence of biomarkers \\nthat are unaffected by patients' low physical activity level. Our analysis used \\nsemi-parametric efficient estimators, an initial Super Learner fit followed by a \\none-step correction, three mediators, and natural direct and indirect estimands, \\nto decompose the average effect of ME/CFS status on molecular and cellular \\ntraits. For this, we used UK Biobank data for up to 1455 ME/CFS cases and \\n131,303 controls. Hundreds of traits differed significantly between cases and \\ncontrols, including 116 significant for both female and male cohorts. These were \\nindicative of chronic inflammation, insulin resistance and liver disease. Nine \\nof 14 traits were replicated in the smaller All-of-Us cohort. Results cannot be \\nexplained by restricted activity: via an activity mediator, ME/CFS status \\nsignificantly affected only 1 of 3237 traits. Individuals with post-exertional \\nmalaise show stronger biomarker differences. Single traits could not cleanly \\ndistinguish cases from controls. Nevertheless, these results keep alive the \\nfuture ambition of a blood-based biomarker panel for accurate ME/CFS diagnosis.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1038/s44321-025-00258-8\\nPMID: 40537675\\n\\nConflict of interest statement: Disclosure and competing interests statement. \\nThe authors declare no competing interests.\",\n",
       " \"1. Discov Oncol. 2025 Jun 20;16(1):1160. doi: 10.1007/s12672-025-02710-0.\\n\\nFAM64A silencing inhibits the proliferation, migration, invasion, and \\nepithelial-mesenchymal transition in ovarian cancer cells via activating Hippo \\npathway.\\n\\nLuo J(1), Li R(2).\\n\\nAuthor information:\\n(1)Department of Gynecology, Ganzhou People's Hospital, Ganzhou Hospital-Nanfang \\nHospital, The Affiliated Ganzhou Hospital, Jiangxi Medical College, Southern \\nMedical University, Nanchang University, Ganzhou, Jiangxi, China.\\n(2)Department of Oncology, The Third People's Hospital of Jiujiang City, No. \\n408, Shili Road, Jiujiang, 332000, Jiangxi, China. liruiyang26@126.com.\\n\\nOBJECTIVE: Ovarian cancer (OC) is a highly aggressive malignancy in females. We \\naim to investigate the potential gene target and examine its impact on OC.\\nMETHODS: Hub genes were determined using protein-protein interaction networks \\nbased on differently expressed genes in GSE12470 and GSE14407 datasets. The \\nimpact of FAM64A on the malignant phenotype of OC cells was evaluated by cell \\ncounting kit-8, 5-ethynyl-2'-deoxyuridine staining, wound healing, and transwell \\nassays. The epithelial-mesenchymal transition (EMT) process was assessed by \\ndetermining the protein expression of E-cadherin, N-cadherin, and Vimentin.\\nRESULTS: We identified the 18 hub genes of OC with substantial predictive value. \\nFAM64A was selected as a candidate gene. The silencing of FAM64A suppressed the \\nviability (si-NC: 0.78\\u2009±\\u20090.04, 0.95\\u2009±\\u20090.08; si-FAM64A: 0.58\\u2009±\\u20090.05, \\n0.64\\u2009±\\u20090.11), proliferation (si-NC: 100.00\\u2009±\\u20099.36, 100.00\\u2009±\\u200914.70; si-FAM64A: \\n34.79\\u2009±\\u20098.88, 44.55\\u2009±\\u20094.91), migration (si-NC: 61.92\\u2009±\\u20098.06, 60.08\\u2009±\\u20095.22; \\nsi-FAM64A: 45.88\\u2009±\\u20098.36, 37.78\\u2009±\\u20097.29), and invasion (si-NC: 130.00\\u2009±\\u200910.34, \\n144.00\\u2009±\\u200913.40; si-FAM64A: 81.00\\u2009±\\u200916.99, 115.60\\u2009±\\u200913.30) of A2780 and SKOV3 \\ncells. FAM64A silencing reduced the EMT in OC cells. The Hippo pathway was \\nidentified as the central pathway implicated in the regulatory role of FAM64A in \\nOC. The silencing of FAM64A caused an increase in the protein expression within \\nthe Hippo pathway in both A2780 and SKOV3 cells.\\nCONCLUSION: Knockdown of FAM64A emerges as a promising therapeutic target for \\nOC, exerting an inhibitory role in OC by activating the Hippo pathway.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s12672-025-02710-0\\nPMID: 40537674\\n\\nConflict of interest statement: Declarations. Ethics approval and consent to \\nparticipate: Not applicable. Consent for publication: Not applicable. Competing \\ninterests: The authors declare that they have no competing interests.\",\n",
       " '1. J Med Ultrason (2001). 2025 Jun 19. doi: 10.1007/s10396-025-01561-0. Online \\nahead of print.\\n\\nThe emerging future of breast ultrasound: a strategic modality amid population \\ndecline and healthcare system strain in Japan.\\n\\nNakashima K(1).\\n\\nAuthor information:\\n(1)Division of Breast Imaging and Breast Interventional Radiology, Shizuoka \\nCancer Center Hospital, Shizuoka, Japan. kaz.nakashima@scchr.jp.\\n\\nDOI: 10.1007/s10396-025-01561-0\\nPMID: 40537673',\n",
       " '1. Biochem Genet. 2025 Jun 19. doi: 10.1007/s10528-025-11159-3. Online ahead of \\nprint.\\n\\nMolecular Study of the Poly (ADP-ribose) Polymerase-1 Gene as a Promotor of \\nInflammation-Driven Colorectal Carcinoma.\\n\\nAziz EHH(1), Magdy A(2), Abo Zaid M(3), Assaf RH(1).\\n\\nAuthor information:\\n(1)Department of Medical Biochemistry and Molecular Biology, Faculty of \\nMedicine, Mansoura University, Mansoura, Egypt.\\n(2)Department of Medical Biochemistry and Molecular Biology, Faculty of \\nMedicine, Mansoura University, Mansoura, Egypt. alshimaamagdy82@yahoo.com.\\n(3)Gastroenterology Surgical Center, Faculty of Medicine, Mansoura University, \\nMansoura, Egypt.\\n\\nColorectal cancer (CRC) is the third most common cancer and the second leading \\ncause of cancer-related deaths worldwide. Chronic inflammation is a risk factor \\nfor various cancers, including CRC. However, the specific mechanisms by which \\ninflammation contributes to cancer development are not fully understood. Our \\nstudy assessed PARP1 and NF-κB mRNA expression in different stages of CRC, \\naiming to elucidate their roles in inflammation-driven CRC pathogenesis and \\ndefine their stage-specific expression patterns. The study involved 35 CRC \\ntissue samples and a control group of 25 samples from the healthy margins of \\ncolon cancer.\\xa0PARP1 and NF-κB mRNA levels were determined by reverse \\ntranscription-quantitative polymerase chain reaction (RT-qPCR) in tumor tissue \\nsamples, as well as the adjacent part of normal tissue. Our results revealed \\nthat PARP1 and NF-κB/p50 gene expression was significantly higher in CRC vs. \\ncontrol. Furthermore, there was a positive correlation between PARP1 and \\nNF-κB/p50 mRNA expression levels. PARP1 was found to be responsible for 14.5% of \\nthe change in NF-κB/p50. Both PARP1 and NF-κB/p50 had high accuracy in the \\ndiagnosis of CRC with AUC\\u2009=\\u20090.905 and 0.956, respectively. This study revealed \\nthe overexpression of PARP1 and NF-κB genes in CRC cases, which suggests that \\nthe use of PARP1 inhibitors and anti-inflammatory drugs could be effective in \\nCRC treatment. PARP1/NF-κB showed preliminary associations with CRC that merit \\ndiagnostic evaluation in larger studies. Our data suggest that PARP1 and NF-κB \\nexpression may complement CEA in characterizing CRC biology though future \\nstudies must determine whether these markers have independent prognostic value.\\n\\n© 2025. The Author(s), under exclusive licence to Springer Science+Business \\nMedia, LLC, part of Springer Nature.\\n\\nDOI: 10.1007/s10528-025-11159-3\\nPMID: 40537666\\n\\nConflict of interest statement: Declarations. Conflict of interest: The authors \\ndeclare that they have no conflicts of interest. Ethical Approval and Consent to \\nParticipate: This study was approved by the Institutional Review Board (IRB) \\nCommittee of Mansoura University, Faculty of Medicine (code number \\nMS.19.05.649). Informed Consent: Signed informed consent was obtained from all \\nparticipants in this study.',\n",
       " '1. Nat Rev Genet. 2025 Jun 19. doi: 10.1038/s41576-025-00859-6. Online ahead of \\nprint.\\n\\nRNA polymerase II transcription compartments - from factories to condensates.\\n\\nRippe K(1)(2), Papantonis A(3).\\n\\nAuthor information:\\n(1)Division of Chromatin Networks, German Cancer Research Center (DKFZ), \\nHeidelberg, Germany. karsten.rippe@dkfz.de.\\n(2)Center for Quantitative Analysis of Molecular and Cellular Biosystems \\n(BioQuant), University of Heidelberg, Heidelberg, Germany. \\nkarsten.rippe@dkfz.de.\\n(3)Translational Epigenetics and Genome Architecture Group, Institute of \\nPathology, University Medical Center Göttingen, Göttingen, Germany. \\nargyris.papantonis@med.uni-goettingen.de.\\n\\nTranscription by RNA polymerase II is a fundamental step in gene regulation that \\nmainly occurs in discrete nuclear foci, or transcription compartments, \\ncharacterized by a high local concentration of polymerases and nascent RNA. \\nEarly studies referred to these foci as transcription factories, proposing that \\nthey harbour most transcriptional activity and all relevant protein machinery to \\nproduce mature RNAs. However, this model of transcriptional organization has \\nlong remained controversial owing to its mechanistic uncertainties. Recently, \\nnew insights into how these foci may form are being provided by studies of \\nphase-separated transcriptional condensates that encompass RNA polymerases, \\ntranscription factors and RNA. Advances in 3D genomics and chromatin imaging are \\nalso deepening our understanding of how transcription compartments might \\nfacilitate communication between cis-regulatory elements in 3D nuclear space. In \\nthis Review, we contrast historical work on transcription factories with recent \\nfindings on transcriptional condensates to better understand the architecture \\nand functional relevance of transcription compartments.\\n\\n© 2025. Springer Nature Limited.\\n\\nDOI: 10.1038/s41576-025-00859-6\\nPMID: 40537661\\n\\nConflict of interest statement: Competing interests: The authors declare no \\ncompeting interests.',\n",
       " '1. Probiotics Antimicrob Proteins. 2025 Jun 20. doi: 10.1007/s12602-025-10631-1. \\nOnline ahead of print.\\n\\nEmerging Synbiotics Consisting of Catechin and Probiotic Bacteria: Exploring \\nAggregation, Adhesion, Antioxidant, and Anticancer Effects.\\n\\nDavran Bulut S(1), Derebası BN(1), Günay S(2), Erden Y(3), Celebioglu HU(4), \\nÜnal MÜ(5).\\n\\nAuthor information:\\n(1)Department of Biotechnology, Faculty of Science, Bartin University, Bartin, \\nTurkey.\\n(2)Department of Biophysics, Faculty of Medicine, Gazi University, Ankara, \\nTurkey.\\n(3)Department of Molecular Biology and Genetics, Faculty of Science, Bartin \\nUniversity, Bartin, Turkey.\\n(4)Department of Biotechnology, Faculty of Science, Bartin University, Bartin, \\nTurkey. ufukcelebioglu@gmail.com.\\n(5)Department of Food Engineering, Faculty of Engineering, University of \\nCukurova, Balcali, Adana, Turkey.\\n\\nProbiotic bacteria are microorganisms that confer health benefits on the host, \\nwhen administered in adequate amounts. Catechin, naturally found in nature and \\nhuman diet, has strong antioxidant, antibacterial, and anticancer potentials. \\nPhenolic compounds and probiotic bacteria are found together in gastrointestinal \\ntract (GIT), and their potential synbiotic interactions are not fully \\nunderstood. The present study aims to investigate the probiotic properties \\neffects of catechin on (bacterial growth, auto-aggregation, co-aggregation) on \\nwell-known probiotic bacteria Lactobacillus acidophilus LA-5 (LA-5) and \\nLacticaseibacillus rhamnosus GG (LGG), and in vitro cytotoxic effects of the \\nsynbiotic combination of catechin and probiotic bacteria on colon cancer cells \\nCaco-2. Results show that catechin contributed to bacterial growth of LA-5, \\nauto-aggregation of LA-5, co-aggregation of LA-5 with Staphylococcus aureus. \\nCatechin increased the adhesion potential of LA-5 and LGG by 6% and 5%, \\nrespectively. In DPPH scavenging activity, LGG grown with catechin exerted 30% \\nmore antioxidant potential compared the LGG. Moreover, LA-5 grown in the \\npresence of catechin led to higher cell death of Caco-2 when compared to control \\nwhile it did not influence the genotoxicity property of LA-5. However, catechin \\ndid not promote the cytotoxicity and genotoxicity properties of LGG. In \\nconclusion, catechin enhanced many physiological properties of probiotic \\nbacteria and thus, synbiotic relationship of catechin and probiotics may offer \\nnovel insights into the prevention/treatment of colon cancer.\\n\\n© 2025. The Author(s).\\n\\nDOI: 10.1007/s12602-025-10631-1\\nPMID: 40537614\\n\\nConflict of interest statement: Declarations. Competing interests: The authors \\ndeclare no competing interests.']"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "fetch_abstracts(disease=\"cancer\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "caa84990",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/venkateshmunaga/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    },
    {
     "ename": "OSError",
     "evalue": "d4data/biobert-chemical-ner is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'\nIf this is a private repository, make sure to pass a token having permission to this repo either by logging in with `huggingface-cli login` or by passing `token=<your_token>`",
     "output_type": "error",
     "traceback": [
      "\u001b[31m---------------------------------------------------------------------------\u001b[39m",
      "\u001b[31mHTTPError\u001b[39m                                 Traceback (most recent call last)",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/utils/_http.py:409\u001b[39m, in \u001b[36mhf_raise_for_status\u001b[39m\u001b[34m(response, endpoint_name)\u001b[39m\n\u001b[32m    408\u001b[39m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m--> \u001b[39m\u001b[32m409\u001b[39m     \u001b[43mresponse\u001b[49m\u001b[43m.\u001b[49m\u001b[43mraise_for_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    410\u001b[39m \u001b[38;5;28;01mexcept\u001b[39;00m HTTPError \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/requests/models.py:1026\u001b[39m, in \u001b[36mResponse.raise_for_status\u001b[39m\u001b[34m(self)\u001b[39m\n\u001b[32m   1025\u001b[39m \u001b[38;5;28;01mif\u001b[39;00m http_error_msg:\n\u001b[32m-> \u001b[39m\u001b[32m1026\u001b[39m     \u001b[38;5;28;01mraise\u001b[39;00m HTTPError(http_error_msg, response=\u001b[38;5;28mself\u001b[39m)\n",
      "\u001b[31mHTTPError\u001b[39m: 401 Client Error: Unauthorized for url: https://huggingface.co/d4data/biobert-chemical-ner/resolve/main/tokenizer_config.json",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[31mRepositoryNotFoundError\u001b[39m                   Traceback (most recent call last)",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/transformers/utils/hub.py:470\u001b[39m, in \u001b[36mcached_files\u001b[39m\u001b[34m(path_or_repo_id, filenames, cache_dir, force_download, resume_download, proxies, token, revision, local_files_only, subfolder, repo_type, user_agent, _raise_exceptions_for_gated_repo, _raise_exceptions_for_missing_entries, _raise_exceptions_for_connection_errors, _commit_hash, **deprecated_kwargs)\u001b[39m\n\u001b[32m    468\u001b[39m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mlen\u001b[39m(full_filenames) == \u001b[32m1\u001b[39m:\n\u001b[32m    469\u001b[39m     \u001b[38;5;66;03m# This is slightly better for only 1 file\u001b[39;00m\n\u001b[32m--> \u001b[39m\u001b[32m470\u001b[39m     \u001b[43mhf_hub_download\u001b[49m\u001b[43m(\u001b[49m\n\u001b[32m    471\u001b[39m \u001b[43m        \u001b[49m\u001b[43mpath_or_repo_id\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    472\u001b[39m \u001b[43m        \u001b[49m\u001b[43mfilenames\u001b[49m\u001b[43m[\u001b[49m\u001b[32;43m0\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    473\u001b[39m \u001b[43m        \u001b[49m\u001b[43msubfolder\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43;01mNone\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mif\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;28;43mlen\u001b[39;49m\u001b[43m(\u001b[49m\u001b[43msubfolder\u001b[49m\u001b[43m)\u001b[49m\u001b[43m \u001b[49m\u001b[43m==\u001b[49m\u001b[43m \u001b[49m\u001b[32;43m0\u001b[39;49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01melse\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43msubfolder\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    474\u001b[39m \u001b[43m        \u001b[49m\u001b[43mrepo_type\u001b[49m\u001b[43m=\u001b[49m\u001b[43mrepo_type\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    475\u001b[39m \u001b[43m        \u001b[49m\u001b[43mrevision\u001b[49m\u001b[43m=\u001b[49m\u001b[43mrevision\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    476\u001b[39m \u001b[43m        \u001b[49m\u001b[43mcache_dir\u001b[49m\u001b[43m=\u001b[49m\u001b[43mcache_dir\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    477\u001b[39m \u001b[43m        \u001b[49m\u001b[43muser_agent\u001b[49m\u001b[43m=\u001b[49m\u001b[43muser_agent\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    478\u001b[39m \u001b[43m        \u001b[49m\u001b[43mforce_download\u001b[49m\u001b[43m=\u001b[49m\u001b[43mforce_download\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    479\u001b[39m \u001b[43m        \u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m=\u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    480\u001b[39m \u001b[43m        \u001b[49m\u001b[43mresume_download\u001b[49m\u001b[43m=\u001b[49m\u001b[43mresume_download\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    481\u001b[39m \u001b[43m        \u001b[49m\u001b[43mtoken\u001b[49m\u001b[43m=\u001b[49m\u001b[43mtoken\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    482\u001b[39m \u001b[43m        \u001b[49m\u001b[43mlocal_files_only\u001b[49m\u001b[43m=\u001b[49m\u001b[43mlocal_files_only\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    483\u001b[39m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    484\u001b[39m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/utils/_validators.py:114\u001b[39m, in \u001b[36mvalidate_hf_hub_args.<locals>._inner_fn\u001b[39m\u001b[34m(*args, **kwargs)\u001b[39m\n\u001b[32m    112\u001b[39m     kwargs = smoothly_deprecate_use_auth_token(fn_name=fn.\u001b[34m__name__\u001b[39m, has_token=has_token, kwargs=kwargs)\n\u001b[32m--> \u001b[39m\u001b[32m114\u001b[39m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mfn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m*\u001b[49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m*\u001b[49m\u001b[43m*\u001b[49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py:1008\u001b[39m, in \u001b[36mhf_hub_download\u001b[39m\u001b[34m(repo_id, filename, subfolder, repo_type, revision, library_name, library_version, cache_dir, local_dir, user_agent, force_download, proxies, etag_timeout, token, local_files_only, headers, endpoint, resume_download, force_filename, local_dir_use_symlinks)\u001b[39m\n\u001b[32m   1007\u001b[39m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[32m-> \u001b[39m\u001b[32m1008\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43m_hf_hub_download_to_cache_dir\u001b[49m\u001b[43m(\u001b[49m\n\u001b[32m   1009\u001b[39m \u001b[43m        \u001b[49m\u001b[38;5;66;43;03m# Destination\u001b[39;49;00m\n\u001b[32m   1010\u001b[39m \u001b[43m        \u001b[49m\u001b[43mcache_dir\u001b[49m\u001b[43m=\u001b[49m\u001b[43mcache_dir\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1011\u001b[39m \u001b[43m        \u001b[49m\u001b[38;5;66;43;03m# File info\u001b[39;49;00m\n\u001b[32m   1012\u001b[39m \u001b[43m        \u001b[49m\u001b[43mrepo_id\u001b[49m\u001b[43m=\u001b[49m\u001b[43mrepo_id\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1013\u001b[39m \u001b[43m        \u001b[49m\u001b[43mfilename\u001b[49m\u001b[43m=\u001b[49m\u001b[43mfilename\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1014\u001b[39m \u001b[43m        \u001b[49m\u001b[43mrepo_type\u001b[49m\u001b[43m=\u001b[49m\u001b[43mrepo_type\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1015\u001b[39m \u001b[43m        \u001b[49m\u001b[43mrevision\u001b[49m\u001b[43m=\u001b[49m\u001b[43mrevision\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1016\u001b[39m \u001b[43m        \u001b[49m\u001b[38;5;66;43;03m# HTTP info\u001b[39;49;00m\n\u001b[32m   1017\u001b[39m \u001b[43m        \u001b[49m\u001b[43mendpoint\u001b[49m\u001b[43m=\u001b[49m\u001b[43mendpoint\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1018\u001b[39m \u001b[43m        \u001b[49m\u001b[43metag_timeout\u001b[49m\u001b[43m=\u001b[49m\u001b[43metag_timeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1019\u001b[39m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[43m=\u001b[49m\u001b[43mhf_headers\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1020\u001b[39m \u001b[43m        \u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m=\u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1021\u001b[39m \u001b[43m        \u001b[49m\u001b[43mtoken\u001b[49m\u001b[43m=\u001b[49m\u001b[43mtoken\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1022\u001b[39m \u001b[43m        \u001b[49m\u001b[38;5;66;43;03m# Additional options\u001b[39;49;00m\n\u001b[32m   1023\u001b[39m \u001b[43m        \u001b[49m\u001b[43mlocal_files_only\u001b[49m\u001b[43m=\u001b[49m\u001b[43mlocal_files_only\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1024\u001b[39m \u001b[43m        \u001b[49m\u001b[43mforce_download\u001b[49m\u001b[43m=\u001b[49m\u001b[43mforce_download\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1025\u001b[39m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py:1115\u001b[39m, in \u001b[36m_hf_hub_download_to_cache_dir\u001b[39m\u001b[34m(cache_dir, repo_id, filename, repo_type, revision, endpoint, etag_timeout, headers, proxies, token, local_files_only, force_download)\u001b[39m\n\u001b[32m   1114\u001b[39m     \u001b[38;5;66;03m# Otherwise, raise appropriate error\u001b[39;00m\n\u001b[32m-> \u001b[39m\u001b[32m1115\u001b[39m     \u001b[43m_raise_on_head_call_error\u001b[49m\u001b[43m(\u001b[49m\u001b[43mhead_call_error\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mforce_download\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mlocal_files_only\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m   1117\u001b[39m \u001b[38;5;66;03m# From now on, etag, commit_hash, url and size are not None.\u001b[39;00m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py:1645\u001b[39m, in \u001b[36m_raise_on_head_call_error\u001b[39m\u001b[34m(head_call_error, force_download, local_files_only)\u001b[39m\n\u001b[32m   1640\u001b[39m \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(head_call_error, (RepositoryNotFoundError, GatedRepoError)) \u001b[38;5;129;01mor\u001b[39;00m (\n\u001b[32m   1641\u001b[39m     \u001b[38;5;28misinstance\u001b[39m(head_call_error, HfHubHTTPError) \u001b[38;5;129;01mand\u001b[39;00m head_call_error.response.status_code == \u001b[32m401\u001b[39m\n\u001b[32m   1642\u001b[39m ):\n\u001b[32m   1643\u001b[39m     \u001b[38;5;66;03m# Repo not found or gated => let's raise the actual error\u001b[39;00m\n\u001b[32m   1644\u001b[39m     \u001b[38;5;66;03m# Unauthorized => likely a token issue => let's raise the actual error\u001b[39;00m\n\u001b[32m-> \u001b[39m\u001b[32m1645\u001b[39m     \u001b[38;5;28;01mraise\u001b[39;00m head_call_error\n\u001b[32m   1646\u001b[39m \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[32m   1647\u001b[39m     \u001b[38;5;66;03m# Otherwise: most likely a connection issue or Hub downtime => let's warn the user\u001b[39;00m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py:1533\u001b[39m, in \u001b[36m_get_metadata_or_catch_error\u001b[39m\u001b[34m(repo_id, filename, repo_type, revision, endpoint, proxies, etag_timeout, headers, token, local_files_only, relative_filename, storage_folder)\u001b[39m\n\u001b[32m   1532\u001b[39m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[32m-> \u001b[39m\u001b[32m1533\u001b[39m     metadata = \u001b[43mget_hf_file_metadata\u001b[49m\u001b[43m(\u001b[49m\n\u001b[32m   1534\u001b[39m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[43m=\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m=\u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[43m=\u001b[49m\u001b[43metag_timeout\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheaders\u001b[49m\u001b[43m=\u001b[49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mtoken\u001b[49m\u001b[43m=\u001b[49m\u001b[43mtoken\u001b[49m\n\u001b[32m   1535\u001b[39m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m   1536\u001b[39m \u001b[38;5;28;01mexcept\u001b[39;00m EntryNotFoundError \u001b[38;5;28;01mas\u001b[39;00m http_error:\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/utils/_validators.py:114\u001b[39m, in \u001b[36mvalidate_hf_hub_args.<locals>._inner_fn\u001b[39m\u001b[34m(*args, **kwargs)\u001b[39m\n\u001b[32m    112\u001b[39m     kwargs = smoothly_deprecate_use_auth_token(fn_name=fn.\u001b[34m__name__\u001b[39m, has_token=has_token, kwargs=kwargs)\n\u001b[32m--> \u001b[39m\u001b[32m114\u001b[39m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mfn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m*\u001b[49m\u001b[43margs\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m*\u001b[49m\u001b[43m*\u001b[49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py:1450\u001b[39m, in \u001b[36mget_hf_file_metadata\u001b[39m\u001b[34m(url, token, proxies, timeout, library_name, library_version, user_agent, headers)\u001b[39m\n\u001b[32m   1449\u001b[39m \u001b[38;5;66;03m# Retrieve metadata\u001b[39;00m\n\u001b[32m-> \u001b[39m\u001b[32m1450\u001b[39m r = \u001b[43m_request_wrapper\u001b[49m\u001b[43m(\u001b[49m\n\u001b[32m   1451\u001b[39m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m=\u001b[49m\u001b[33;43m\"\u001b[39;49m\u001b[33;43mHEAD\u001b[39;49m\u001b[33;43m\"\u001b[39;49m\u001b[43m,\u001b[49m\n\u001b[32m   1452\u001b[39m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m=\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1453\u001b[39m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[43m=\u001b[49m\u001b[43mhf_headers\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1454\u001b[39m \u001b[43m    \u001b[49m\u001b[43mallow_redirects\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[32m   1455\u001b[39m \u001b[43m    \u001b[49m\u001b[43mfollow_relative_redirects\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43;01mTrue\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[32m   1456\u001b[39m \u001b[43m    \u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m=\u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1457\u001b[39m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[43m=\u001b[49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m   1458\u001b[39m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m   1459\u001b[39m hf_raise_for_status(r)\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py:286\u001b[39m, in \u001b[36m_request_wrapper\u001b[39m\u001b[34m(method, url, follow_relative_redirects, **params)\u001b[39m\n\u001b[32m    285\u001b[39m \u001b[38;5;28;01mif\u001b[39;00m follow_relative_redirects:\n\u001b[32m--> \u001b[39m\u001b[32m286\u001b[39m     response = \u001b[43m_request_wrapper\u001b[49m\u001b[43m(\u001b[49m\n\u001b[32m    287\u001b[39m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m=\u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    288\u001b[39m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[43m=\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    289\u001b[39m \u001b[43m        \u001b[49m\u001b[43mfollow_relative_redirects\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[32m    290\u001b[39m \u001b[43m        \u001b[49m\u001b[43m*\u001b[49m\u001b[43m*\u001b[49m\u001b[43mparams\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    291\u001b[39m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    293\u001b[39m     \u001b[38;5;66;03m# If redirection, we redirect only relative paths.\u001b[39;00m\n\u001b[32m    294\u001b[39m     \u001b[38;5;66;03m# This is useful in case of a renamed repository.\u001b[39;00m\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/file_download.py:310\u001b[39m, in \u001b[36m_request_wrapper\u001b[39m\u001b[34m(method, url, follow_relative_redirects, **params)\u001b[39m\n\u001b[32m    309\u001b[39m response = http_backoff(method=method, url=url, **params, retry_on_exceptions=(), retry_on_status_codes=(\u001b[32m429\u001b[39m,))\n\u001b[32m--> \u001b[39m\u001b[32m310\u001b[39m \u001b[43mhf_raise_for_status\u001b[49m\u001b[43m(\u001b[49m\u001b[43mresponse\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    311\u001b[39m \u001b[38;5;28;01mreturn\u001b[39;00m response\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/huggingface_hub/utils/_http.py:459\u001b[39m, in \u001b[36mhf_raise_for_status\u001b[39m\u001b[34m(response, endpoint_name)\u001b[39m\n\u001b[32m    450\u001b[39m     message = (\n\u001b[32m    451\u001b[39m         \u001b[33mf\u001b[39m\u001b[33m\"\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mresponse.status_code\u001b[38;5;132;01m}\u001b[39;00m\u001b[33m Client Error.\u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    452\u001b[39m         + \u001b[33m\"\u001b[39m\u001b[38;5;130;01m\\n\u001b[39;00m\u001b[38;5;130;01m\\n\u001b[39;00m\u001b[33m\"\u001b[39m\n\u001b[32m   (...)\u001b[39m\u001b[32m    457\u001b[39m         \u001b[33m\"\u001b[39m\u001b[33m https://huggingface.co/docs/huggingface_hub/authentication\u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    458\u001b[39m     )\n\u001b[32m--> \u001b[39m\u001b[32m459\u001b[39m     \u001b[38;5;28;01mraise\u001b[39;00m _format(RepositoryNotFoundError, message, response) \u001b[38;5;28;01mfrom\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01me\u001b[39;00m\n\u001b[32m    461\u001b[39m \u001b[38;5;28;01melif\u001b[39;00m response.status_code == \u001b[32m400\u001b[39m:\n",
      "\u001b[31mRepositoryNotFoundError\u001b[39m: 401 Client Error. (Request ID: Root=1-68550e34-6bf338ff5b7632ef230d4b27;8638375e-441f-4b06-a00e-c0126761a539)\n\nRepository Not Found for url: https://huggingface.co/d4data/biobert-chemical-ner/resolve/main/tokenizer_config.json.\nPlease make sure you specified the correct `repo_id` and `repo_type`.\nIf you are trying to access a private or gated repo, make sure you are authenticated. For more details, see https://huggingface.co/docs/huggingface_hub/authentication\nInvalid username or password.",
      "\nThe above exception was the direct cause of the following exception:\n",
      "\u001b[31mOSError\u001b[39m                                   Traceback (most recent call last)",
      "\u001b[36mCell\u001b[39m\u001b[36m \u001b[39m\u001b[32mIn[13]\u001b[39m\u001b[32m, line 5\u001b[39m\n\u001b[32m      2\u001b[39m \u001b[38;5;28;01mfrom\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01mtransformers\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[38;5;28;01mimport\u001b[39;00m AutoTokenizer, AutoModelForTokenClassification\n\u001b[32m      3\u001b[39m \u001b[38;5;28;01mfrom\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01mtransformers\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[38;5;28;01mimport\u001b[39;00m pipeline\n\u001b[32m----> \u001b[39m\u001b[32m5\u001b[39m tokenizer = \u001b[43mAutoTokenizer\u001b[49m\u001b[43m.\u001b[49m\u001b[43mfrom_pretrained\u001b[49m\u001b[43m(\u001b[49m\u001b[33;43m\"\u001b[39;49m\u001b[33;43md4data/biobert-chemical-ner\u001b[39;49m\u001b[33;43m\"\u001b[39;49m\u001b[43m)\u001b[49m\n\u001b[32m      6\u001b[39m model = AutoModelForTokenClassification.from_pretrained(\u001b[33m\"\u001b[39m\u001b[33md4data/biobert-chemical-ner\u001b[39m\u001b[33m\"\u001b[39m)\n\u001b[32m      8\u001b[39m ner = pipeline(\u001b[33m\"\u001b[39m\u001b[33mner\u001b[39m\u001b[33m\"\u001b[39m, model=model, tokenizer=tokenizer, aggregation_strategy=\u001b[33m\"\u001b[39m\u001b[33msimple\u001b[39m\u001b[33m\"\u001b[39m)\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/transformers/models/auto/tokenization_auto.py:950\u001b[39m, in \u001b[36mAutoTokenizer.from_pretrained\u001b[39m\u001b[34m(cls, pretrained_model_name_or_path, *inputs, **kwargs)\u001b[39m\n\u001b[32m    947\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m tokenizer_class.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)\n\u001b[32m    949\u001b[39m \u001b[38;5;66;03m# Next, let's try to use the tokenizer_config file to get the tokenizer class.\u001b[39;00m\n\u001b[32m--> \u001b[39m\u001b[32m950\u001b[39m tokenizer_config = \u001b[43mget_tokenizer_config\u001b[49m\u001b[43m(\u001b[49m\u001b[43mpretrained_model_name_or_path\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m*\u001b[49m\u001b[43m*\u001b[49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    951\u001b[39m \u001b[38;5;28;01mif\u001b[39;00m \u001b[33m\"\u001b[39m\u001b[33m_commit_hash\u001b[39m\u001b[33m\"\u001b[39m \u001b[38;5;129;01min\u001b[39;00m tokenizer_config:\n\u001b[32m    952\u001b[39m     kwargs[\u001b[33m\"\u001b[39m\u001b[33m_commit_hash\u001b[39m\u001b[33m\"\u001b[39m] = tokenizer_config[\u001b[33m\"\u001b[39m\u001b[33m_commit_hash\u001b[39m\u001b[33m\"\u001b[39m]\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/transformers/models/auto/tokenization_auto.py:782\u001b[39m, in \u001b[36mget_tokenizer_config\u001b[39m\u001b[34m(pretrained_model_name_or_path, cache_dir, force_download, resume_download, proxies, token, revision, local_files_only, subfolder, **kwargs)\u001b[39m\n\u001b[32m    779\u001b[39m     token = use_auth_token\n\u001b[32m    781\u001b[39m commit_hash = kwargs.get(\u001b[33m\"\u001b[39m\u001b[33m_commit_hash\u001b[39m\u001b[33m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m)\n\u001b[32m--> \u001b[39m\u001b[32m782\u001b[39m resolved_config_file = \u001b[43mcached_file\u001b[49m\u001b[43m(\u001b[49m\n\u001b[32m    783\u001b[39m \u001b[43m    \u001b[49m\u001b[43mpretrained_model_name_or_path\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    784\u001b[39m \u001b[43m    \u001b[49m\u001b[43mTOKENIZER_CONFIG_FILE\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    785\u001b[39m \u001b[43m    \u001b[49m\u001b[43mcache_dir\u001b[49m\u001b[43m=\u001b[49m\u001b[43mcache_dir\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    786\u001b[39m \u001b[43m    \u001b[49m\u001b[43mforce_download\u001b[49m\u001b[43m=\u001b[49m\u001b[43mforce_download\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    787\u001b[39m \u001b[43m    \u001b[49m\u001b[43mresume_download\u001b[49m\u001b[43m=\u001b[49m\u001b[43mresume_download\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    788\u001b[39m \u001b[43m    \u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m=\u001b[49m\u001b[43mproxies\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    789\u001b[39m \u001b[43m    \u001b[49m\u001b[43mtoken\u001b[49m\u001b[43m=\u001b[49m\u001b[43mtoken\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    790\u001b[39m \u001b[43m    \u001b[49m\u001b[43mrevision\u001b[49m\u001b[43m=\u001b[49m\u001b[43mrevision\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    791\u001b[39m \u001b[43m    \u001b[49m\u001b[43mlocal_files_only\u001b[49m\u001b[43m=\u001b[49m\u001b[43mlocal_files_only\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    792\u001b[39m \u001b[43m    \u001b[49m\u001b[43msubfolder\u001b[49m\u001b[43m=\u001b[49m\u001b[43msubfolder\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    793\u001b[39m \u001b[43m    \u001b[49m\u001b[43m_raise_exceptions_for_gated_repo\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[32m    794\u001b[39m \u001b[43m    \u001b[49m\u001b[43m_raise_exceptions_for_missing_entries\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[32m    795\u001b[39m \u001b[43m    \u001b[49m\u001b[43m_raise_exceptions_for_connection_errors\u001b[49m\u001b[43m=\u001b[49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[32m    796\u001b[39m \u001b[43m    \u001b[49m\u001b[43m_commit_hash\u001b[49m\u001b[43m=\u001b[49m\u001b[43mcommit_hash\u001b[49m\u001b[43m,\u001b[49m\n\u001b[32m    797\u001b[39m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    798\u001b[39m \u001b[38;5;28;01mif\u001b[39;00m resolved_config_file \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[32m    799\u001b[39m     logger.info(\u001b[33m\"\u001b[39m\u001b[33mCould not locate the tokenizer configuration file, will try to use the model config instead.\u001b[39m\u001b[33m\"\u001b[39m)\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/transformers/utils/hub.py:312\u001b[39m, in \u001b[36mcached_file\u001b[39m\u001b[34m(path_or_repo_id, filename, **kwargs)\u001b[39m\n\u001b[32m    254\u001b[39m \u001b[38;5;28;01mdef\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34mcached_file\u001b[39m(\n\u001b[32m    255\u001b[39m     path_or_repo_id: Union[\u001b[38;5;28mstr\u001b[39m, os.PathLike],\n\u001b[32m    256\u001b[39m     filename: \u001b[38;5;28mstr\u001b[39m,\n\u001b[32m    257\u001b[39m     **kwargs,\n\u001b[32m    258\u001b[39m ) -> Optional[\u001b[38;5;28mstr\u001b[39m]:\n\u001b[32m    259\u001b[39m \u001b[38;5;250m    \u001b[39m\u001b[33;03m\"\"\"\u001b[39;00m\n\u001b[32m    260\u001b[39m \u001b[33;03m    Tries to locate a file in a local folder and repo, downloads and cache it if necessary.\u001b[39;00m\n\u001b[32m    261\u001b[39m \n\u001b[32m   (...)\u001b[39m\u001b[32m    310\u001b[39m \u001b[33;03m    ```\u001b[39;00m\n\u001b[32m    311\u001b[39m \u001b[33;03m    \"\"\"\u001b[39;00m\n\u001b[32m--> \u001b[39m\u001b[32m312\u001b[39m     file = \u001b[43mcached_files\u001b[49m\u001b[43m(\u001b[49m\u001b[43mpath_or_repo_id\u001b[49m\u001b[43m=\u001b[49m\u001b[43mpath_or_repo_id\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mfilenames\u001b[49m\u001b[43m=\u001b[49m\u001b[43m[\u001b[49m\u001b[43mfilename\u001b[49m\u001b[43m]\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m*\u001b[49m\u001b[43m*\u001b[49m\u001b[43mkwargs\u001b[49m\u001b[43m)\u001b[49m\n\u001b[32m    313\u001b[39m     file = file[\u001b[32m0\u001b[39m] \u001b[38;5;28;01mif\u001b[39;00m file \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;28;01melse\u001b[39;00m file\n\u001b[32m    314\u001b[39m     \u001b[38;5;28;01mreturn\u001b[39;00m file\n",
      "\u001b[36mFile \u001b[39m\u001b[32m~/Python Practice/BioMarkr/.venv/lib/python3.13/site-packages/transformers/utils/hub.py:502\u001b[39m, in \u001b[36mcached_files\u001b[39m\u001b[34m(path_or_repo_id, filenames, cache_dir, force_download, resume_download, proxies, token, revision, local_files_only, subfolder, repo_type, user_agent, _raise_exceptions_for_gated_repo, _raise_exceptions_for_missing_entries, _raise_exceptions_for_connection_errors, _commit_hash, **deprecated_kwargs)\u001b[39m\n\u001b[32m    499\u001b[39m \u001b[38;5;28;01mexcept\u001b[39;00m \u001b[38;5;167;01mException\u001b[39;00m \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[32m    500\u001b[39m     \u001b[38;5;66;03m# We cannot recover from them\u001b[39;00m\n\u001b[32m    501\u001b[39m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e, RepositoryNotFoundError) \u001b[38;5;129;01mand\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e, GatedRepoError):\n\u001b[32m--> \u001b[39m\u001b[32m502\u001b[39m         \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mOSError\u001b[39;00m(\n\u001b[32m    503\u001b[39m             \u001b[33mf\u001b[39m\u001b[33m\"\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mpath_or_repo_id\u001b[38;5;132;01m}\u001b[39;00m\u001b[33m is not a local folder and is not a valid model identifier \u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    504\u001b[39m             \u001b[33m\"\u001b[39m\u001b[33mlisted on \u001b[39m\u001b[33m'\u001b[39m\u001b[33mhttps://huggingface.co/models\u001b[39m\u001b[33m'\u001b[39m\u001b[38;5;130;01m\\n\u001b[39;00m\u001b[33mIf this is a private repository, make sure to pass a token \u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    505\u001b[39m             \u001b[33m\"\u001b[39m\u001b[33mhaving permission to this repo either by logging in with `huggingface-cli login` or by passing \u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    506\u001b[39m             \u001b[33m\"\u001b[39m\u001b[33m`token=<your_token>`\u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    507\u001b[39m         ) \u001b[38;5;28;01mfrom\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01me\u001b[39;00m\n\u001b[32m    508\u001b[39m     \u001b[38;5;28;01melif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e, RevisionNotFoundError):\n\u001b[32m    509\u001b[39m         \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mOSError\u001b[39;00m(\n\u001b[32m    510\u001b[39m             \u001b[33mf\u001b[39m\u001b[33m\"\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mrevision\u001b[38;5;132;01m}\u001b[39;00m\u001b[33m is not a valid git identifier (branch name, tag name or commit id) that exists \u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    511\u001b[39m             \u001b[33m\"\u001b[39m\u001b[33mfor this model name. Check the model page at \u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    512\u001b[39m             \u001b[33mf\u001b[39m\u001b[33m\"\u001b[39m\u001b[33m'\u001b[39m\u001b[33mhttps://huggingface.co/\u001b[39m\u001b[38;5;132;01m{\u001b[39;00mpath_or_repo_id\u001b[38;5;132;01m}\u001b[39;00m\u001b[33m'\u001b[39m\u001b[33m for available revisions.\u001b[39m\u001b[33m\"\u001b[39m\n\u001b[32m    513\u001b[39m         ) \u001b[38;5;28;01mfrom\u001b[39;00m\u001b[38;5;250m \u001b[39m\u001b[34;01me\u001b[39;00m\n",
      "\u001b[31mOSError\u001b[39m: d4data/biobert-chemical-ner is not a local folder and is not a valid model identifier listed on 'https://huggingface.co/models'\nIf this is a private repository, make sure to pass a token having permission to this repo either by logging in with `huggingface-cli login` or by passing `token=<your_token>`"
     ]
    }
   ],
   "source": [
    "# biomarker_ner.py\n",
    "from transformers import AutoTokenizer, AutoModelForTokenClassification\n",
    "from transformers import pipeline\n",
    "\n",
    "tokenizer = AutoTokenizer.from_pretrained(\"d4data/biobert-chemical-ner\")\n",
    "model = AutoModelForTokenClassification.from_pretrained(\"d4data/biobert-chemical-ner\")\n",
    "\n",
    "ner = pipeline(\"ner\", model=model, tokenizer=tokenizer, aggregation_strategy=\"simple\")\n",
    "\n",
    "def extract_biomarkers(texts):\n",
    "    biomarkers = []\n",
    "    for text in texts:\n",
    "        results = ner(text)\n",
    "        for r in results:\n",
    "            if r[\"entity_group\"] in [\"CHEMICAL\", \"GENE\", \"PROTEIN\"]:\n",
    "                biomarkers.append(r[\"word\"])\n",
    "    return biomarkers\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2356ed5a",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": ".venv",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.13.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
